{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "13720149-e9cc-4f12-8e2d-71a61dba161b",
   "metadata": {},
   "outputs": [
    {
     "ename": "ERROR",
     "evalue": "Error: With R version 3.5 or greater, install Bioconductor packages using BiocManager; see https://bioconductor.org/install\n",
     "output_type": "error",
     "traceback": [
      "Error: With R version 3.5 or greater, install Bioconductor packages using BiocManager; see https://bioconductor.org/install\nTraceback:\n",
      "1. source(\"https://bioconductor.org/biocLite.R\")",
      "2. withVisible(eval(ei, envir))",
      "3. eval(ei, envir)",
      "4. eval(ei, envir)",
      "5. local({\n .     vers <- getRversion()\n .     if (vers >= \"3.6\") {\n .         stop(\"With R version 3.5 or greater, install Bioconductor \", \n .             \"packages using BiocManager; see https://bioconductor.org/install\", \n .             call. = FALSE)\n .     }\n .     biocVers <- tryCatch({\n .         BiocInstaller::biocVersion()\n .     }, error = function(...) {\n .         BioC_version_associated_with_R_version <- get(\".BioC_version_associated_with_R_version\", \n .             envir = asNamespace(\"tools\"), inherits = FALSE)\n .         if (is.function(BioC_version_associated_with_R_version)) \n .             BioC_version_associated_with_R_version()\n .         else BioC_version_associated_with_R_version\n .     })\n .     if (vers < \"3.0\") {\n .         txt <- strwrap(\"A new version of Bioconductor is available\\n            after installing the most recent version of R; see\\n            http://bioconductor.org/install\", \n .             exdent = 2)\n .         message(paste(txt, collapse = \"\\n\"))\n .     }\n .     else if (\"package:BiocInstaller\" %in% search()) {\n .         tryCatch(BiocInstaller:::.onAttach(), error = function(...) NULL)\n .     }\n .     if (vers > \"2.13\" && biocVers > \"2.8\") {\n .         if (exists(\"biocLite\", .GlobalEnv, inherits = FALSE)) {\n .             txt <- strwrap(\"There is an outdated biocLite() function in the\\n                global environment; run 'rm(biocLite)' and try again.\")\n .             stop(\"\\n\", paste(txt, collapse = \"\\n\"))\n .         }\n .         if (!suppressWarnings(require(\"BiocInstaller\", quietly = TRUE))) {\n .             a <- NULL\n .             p <- file.path(Sys.getenv(\"HOME\"), \".R\", \"repositories\")\n .             if (file.exists(p)) {\n .                 a <- tools:::.read_repositories(p)\n .                 if (!\"BioCsoft\" %in% rownames(a)) \n .                   a <- NULL\n .             }\n .             if (is.null(a)) {\n .                 p <- file.path(R.home(\"etc\"), \"repositories\")\n .                 a <- tools:::.read_repositories(p)\n .             }\n .             if (!\"package:utils\" %in% search()) {\n .                 path <- \"//bioconductor.org/biocLite.R\"\n .                 txt <- sprintf(\"use 'source(\\\"https:%s\\\")' or\\n                    'source(\\\"http:%s\\\")' to update 'BiocInstaller' after\\n                    library(\\\"utils\\\")\", \n .                   path, path)\n .                 message(paste(strwrap(txt), collapse = \"\\n  \"))\n .             }\n .             else {\n .                 if (vers >= \"3.2.2\" && vers < \"3.3.0\") {\n .                   con <- file(fl <- tempfile(), \"w\")\n .                   sink(con, type = \"message\")\n .                   tryCatch({\n .                     xx <- close(file(\"https://bioconductor.org\"))\n .                   }, error = function(e) {\n .                     message(conditionMessage(e))\n .                   })\n .                   sink(type = \"message\")\n .                   close(con)\n .                   if (!length(readLines(fl))) \n .                     a[\"BioCsoft\", \"URL\"] <- sub(\"^http:\", \"https:\", \n .                       a[\"BioCsoft\", \"URL\"])\n .                 }\n .                 if (vers >= \"3.5.0\") {\n .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n .                     \"3.7\", a[\"BioCsoft\", \"URL\"])\n .                 }\n .                 else if (vers >= \"3.4.0\") {\n .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n .                     \"3.6\", a[\"BioCsoft\", \"URL\"])\n .                 }\n .                 else if (vers >= \"3.3.0\") {\n .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n .                     \"3.4\", a[\"BioCsoft\", \"URL\"])\n .                 }\n .                 else if (vers >= \"3.2\") {\n .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n .                     \"3.2\", a[\"BioCsoft\", \"URL\"])\n .                 }\n .                 else if (vers == \"3.1.1\") {\n .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n .                     \"3.0\", a[\"BioCsoft\", \"URL\"])\n .                 }\n .                 else if (vers == \"3.1.0\") {\n .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n .                     \"2.14\", a[\"BioCsoft\", \"URL\"])\n .                 }\n .                 else if (vers >= \"2.15\" && vers < \"2.16\") {\n .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n .                     \"2.11\", a[\"BioCsoft\", \"URL\"])\n .                   biocVers <- numeric_version(\"2.11\")\n .                 }\n .                 install.packages(\"BiocInstaller\", repos = a[\"BioCsoft\", \n .                   \"URL\"])\n .                 if (!suppressWarnings(require(\"BiocInstaller\", \n .                   quietly = TRUE))) {\n .                   path0 <- \"//bioconductor.org/packages\"\n .                   path <- sprintf(\"%s/%s/bioc\", path0, as.character(biocVers))\n .                   txt0 <- \"'biocLite.R' failed to install 'BiocInstaller',\\n                        use 'install.packages(\\\"BiocInstaller\\\",\\n                        repos=\\\"https:%s\\\")' or\\n                        'install.packages(\\\"BiocInstaller\\\", repos=\\\"http:%s\\\")'\"\n .                   txt <- sprintf(txt0, path, path)\n .                   message(paste(strwrap(txt), collapse = \"\\n  \"))\n .                 }\n .             }\n .         }\n .         else {\n .             installerVersion <- utils::packageVersion(\"BiocInstaller\")\n .             test0 <- (vers > \"3.1.2\") && !identical(getOption(\"pkgType\"), \n .                 \"source\") && (installerVersion >= \"1.16.0\") && \n .                 (installerVersion <= \"1.16.4\")\n .             if (test0) {\n .                 if (installerVersion < \"1.16.4\") {\n .                   txt <- \"Update BiocInstaller with\\n                         oldPkgType=options(pkgType=\\\"source\\\");\\n                         biocLite(\\\"BiocInstaller\\\"); options(oldPkgType)\"\n .                   message(paste(strwrap(txt, exdent = 2), collapse = \"\\n\"))\n .                 }\n .                 if (vers >= \"3.2\") {\n .                   path <- \"//bioconductor.org/biocLite.R\"\n .                   txt <- sprintf(\"BiocInstaller version %s is too old for\\n                         R version %s; remove.packages(\\\"BiocInstaller\\\"),\\n                         re-start R, then source(\\\"https:%s\\\") or\\n                         source(\\\"http:%s\\\")\", \n .                     biocVers, vers, path, path)\n .                   warning(paste(strwrap(txt, exdent = 2), collapse = \"\\n\"))\n .                 }\n .             }\n .         }\n .     }\n .     else {\n .         tryCatch({\n .             source(\"https://bioconductor.org/getBioC.R\")\n .         }, error = function(e) {\n .             warning(\"https: failed (\", conditionMessage(e), \"), trying http\", \n .                 immediate. = TRUE)\n .             source(\"http://bioconductor.org/getBioC.R\")\n .         })\n .         biocLite <<- function(pkgs, groupName = \"lite\", ...) {\n .             if (missing(pkgs)) \n .                 biocinstall(groupName = groupName, ...)\n .             else biocinstall(pkgs = pkgs, groupName = groupName, \n .                 ...)\n .         }\n .     }\n . })",
      "6. eval.parent(substitute(eval(quote(expr), envir)))",
      "7. eval(expr, p)",
      "8. eval(expr, p)",
      "9. eval(quote({\n .     vers <- getRversion()\n .     if (vers >= \"3.6\") {\n .         stop(\"With R version 3.5 or greater, install Bioconductor \", \n .             \"packages using BiocManager; see https://bioconductor.org/install\", \n .             call. = FALSE)\n .     }\n .     biocVers <- tryCatch({\n .         BiocInstaller::biocVersion()\n .     }, error = function(...) {\n .         BioC_version_associated_with_R_version <- get(\".BioC_version_associated_with_R_version\", \n .             envir = asNamespace(\"tools\"), inherits = FALSE)\n .         if (is.function(BioC_version_associated_with_R_version)) BioC_version_associated_with_R_version() else BioC_version_associated_with_R_version\n .     })\n .     if (vers < \"3.0\") {\n .         txt <- strwrap(\"A new version of Bioconductor is available\\n            after installing the most recent version of R; see\\n            http://bioconductor.org/install\", \n .             exdent = 2)\n .         message(paste(txt, collapse = \"\\n\"))\n .     } else if (\"package:BiocInstaller\" %in% search()) {\n .         tryCatch(BiocInstaller:::.onAttach(), error = function(...) NULL)\n .     }\n .     if (vers > \"2.13\" && biocVers > \"2.8\") {\n .         if (exists(\"biocLite\", .GlobalEnv, inherits = FALSE)) {\n .             txt <- strwrap(\"There is an outdated biocLite() function in the\\n                global environment; run 'rm(biocLite)' and try again.\")\n .             stop(\"\\n\", paste(txt, collapse = \"\\n\"))\n .         }\n .         if (!suppressWarnings(require(\"BiocInstaller\", quietly = TRUE))) {\n .             a <- NULL\n .             p <- file.path(Sys.getenv(\"HOME\"), \".R\", \"repositories\")\n .             if (file.exists(p)) {\n .                 a <- tools:::.read_repositories(p)\n .                 if (!\"BioCsoft\" %in% rownames(a)) a <- NULL\n .             }\n .             if (is.null(a)) {\n .                 p <- file.path(R.home(\"etc\"), \"repositories\")\n .                 a <- tools:::.read_repositories(p)\n .             }\n .             if (!\"package:utils\" %in% search()) {\n .                 path <- \"//bioconductor.org/biocLite.R\"\n .                 txt <- sprintf(\"use 'source(\\\"https:%s\\\")' or\\n                    'source(\\\"http:%s\\\")' to update 'BiocInstaller' after\\n                    library(\\\"utils\\\")\", \n .                   path, path)\n .                 message(paste(strwrap(txt), collapse = \"\\n  \"))\n .             } else {\n .                 if (vers >= \"3.2.2\" && vers < \"3.3.0\") {\n .                   con <- file(fl <- tempfile(), \"w\")\n .                   sink(con, type = \"message\")\n .                   tryCatch({\n .                     xx <- close(file(\"https://bioconductor.org\"))\n .                   }, error = function(e) {\n .                     message(conditionMessage(e))\n .                   })\n .                   sink(type = \"message\")\n .                   close(con)\n .                   if (!length(readLines(fl))) a[\"BioCsoft\", \"URL\"] <- sub(\"^http:\", \n .                     \"https:\", a[\"BioCsoft\", \"URL\"])\n .                 }\n .                 if (vers >= \"3.5.0\") {\n .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n .                     \"3.7\", a[\"BioCsoft\", \"URL\"])\n .                 } else if (vers >= \"3.4.0\") {\n .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n .                     \"3.6\", a[\"BioCsoft\", \"URL\"])\n .                 } else if (vers >= \"3.3.0\") {\n .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n .                     \"3.4\", a[\"BioCsoft\", \"URL\"])\n .                 } else if (vers >= \"3.2\") {\n .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n .                     \"3.2\", a[\"BioCsoft\", \"URL\"])\n .                 } else if (vers == \"3.1.1\") {\n .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n .                     \"3.0\", a[\"BioCsoft\", \"URL\"])\n .                 } else if (vers == \"3.1.0\") {\n .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n .                     \"2.14\", a[\"BioCsoft\", \"URL\"])\n .                 } else if (vers >= \"2.15\" && vers < \"2.16\") {\n .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n .                     \"2.11\", a[\"BioCsoft\", \"URL\"])\n .                   biocVers <- numeric_version(\"2.11\")\n .                 }\n .                 install.packages(\"BiocInstaller\", repos = a[\"BioCsoft\", \n .                   \"URL\"])\n .                 if (!suppressWarnings(require(\"BiocInstaller\", \n .                   quietly = TRUE))) {\n .                   path0 <- \"//bioconductor.org/packages\"\n .                   path <- sprintf(\"%s/%s/bioc\", path0, as.character(biocVers))\n .                   txt0 <- \"'biocLite.R' failed to install 'BiocInstaller',\\n                        use 'install.packages(\\\"BiocInstaller\\\",\\n                        repos=\\\"https:%s\\\")' or\\n                        'install.packages(\\\"BiocInstaller\\\", repos=\\\"http:%s\\\")'\"\n .                   txt <- sprintf(txt0, path, path)\n .                   message(paste(strwrap(txt), collapse = \"\\n  \"))\n .                 }\n .             }\n .         } else {\n .             installerVersion <- utils::packageVersion(\"BiocInstaller\")\n .             test0 <- (vers > \"3.1.2\") && !identical(getOption(\"pkgType\"), \n .                 \"source\") && (installerVersion >= \"1.16.0\") && \n .                 (installerVersion <= \"1.16.4\")\n .             if (test0) {\n .                 if (installerVersion < \"1.16.4\") {\n .                   txt <- \"Update BiocInstaller with\\n                         oldPkgType=options(pkgType=\\\"source\\\");\\n                         biocLite(\\\"BiocInstaller\\\"); options(oldPkgType)\"\n .                   message(paste(strwrap(txt, exdent = 2), collapse = \"\\n\"))\n .                 }\n .                 if (vers >= \"3.2\") {\n .                   path <- \"//bioconductor.org/biocLite.R\"\n .                   txt <- sprintf(\"BiocInstaller version %s is too old for\\n                         R version %s; remove.packages(\\\"BiocInstaller\\\"),\\n                         re-start R, then source(\\\"https:%s\\\") or\\n                         source(\\\"http:%s\\\")\", \n .                     biocVers, vers, path, path)\n .                   warning(paste(strwrap(txt, exdent = 2), collapse = \"\\n\"))\n .                 }\n .             }\n .         }\n .     } else {\n .         tryCatch({\n .             source(\"https://bioconductor.org/getBioC.R\")\n .         }, error = function(e) {\n .             warning(\"https: failed (\", conditionMessage(e), \"), trying http\", \n .                 immediate. = TRUE)\n .             source(\"http://bioconductor.org/getBioC.R\")\n .         })\n .         biocLite <<- function(pkgs, groupName = \"lite\", ...) {\n .             if (missing(pkgs)) biocinstall(groupName = groupName, \n .                 ...) else biocinstall(pkgs = pkgs, groupName = groupName, \n .                 ...)\n .         }\n .     }\n . }), new.env())",
      "10. eval(quote({\n  .     vers <- getRversion()\n  .     if (vers >= \"3.6\") {\n  .         stop(\"With R version 3.5 or greater, install Bioconductor \", \n  .             \"packages using BiocManager; see https://bioconductor.org/install\", \n  .             call. = FALSE)\n  .     }\n  .     biocVers <- tryCatch({\n  .         BiocInstaller::biocVersion()\n  .     }, error = function(...) {\n  .         BioC_version_associated_with_R_version <- get(\".BioC_version_associated_with_R_version\", \n  .             envir = asNamespace(\"tools\"), inherits = FALSE)\n  .         if (is.function(BioC_version_associated_with_R_version)) BioC_version_associated_with_R_version() else BioC_version_associated_with_R_version\n  .     })\n  .     if (vers < \"3.0\") {\n  .         txt <- strwrap(\"A new version of Bioconductor is available\\n            after installing the most recent version of R; see\\n            http://bioconductor.org/install\", \n  .             exdent = 2)\n  .         message(paste(txt, collapse = \"\\n\"))\n  .     } else if (\"package:BiocInstaller\" %in% search()) {\n  .         tryCatch(BiocInstaller:::.onAttach(), error = function(...) NULL)\n  .     }\n  .     if (vers > \"2.13\" && biocVers > \"2.8\") {\n  .         if (exists(\"biocLite\", .GlobalEnv, inherits = FALSE)) {\n  .             txt <- strwrap(\"There is an outdated biocLite() function in the\\n                global environment; run 'rm(biocLite)' and try again.\")\n  .             stop(\"\\n\", paste(txt, collapse = \"\\n\"))\n  .         }\n  .         if (!suppressWarnings(require(\"BiocInstaller\", quietly = TRUE))) {\n  .             a <- NULL\n  .             p <- file.path(Sys.getenv(\"HOME\"), \".R\", \"repositories\")\n  .             if (file.exists(p)) {\n  .                 a <- tools:::.read_repositories(p)\n  .                 if (!\"BioCsoft\" %in% rownames(a)) a <- NULL\n  .             }\n  .             if (is.null(a)) {\n  .                 p <- file.path(R.home(\"etc\"), \"repositories\")\n  .                 a <- tools:::.read_repositories(p)\n  .             }\n  .             if (!\"package:utils\" %in% search()) {\n  .                 path <- \"//bioconductor.org/biocLite.R\"\n  .                 txt <- sprintf(\"use 'source(\\\"https:%s\\\")' or\\n                    'source(\\\"http:%s\\\")' to update 'BiocInstaller' after\\n                    library(\\\"utils\\\")\", \n  .                   path, path)\n  .                 message(paste(strwrap(txt), collapse = \"\\n  \"))\n  .             } else {\n  .                 if (vers >= \"3.2.2\" && vers < \"3.3.0\") {\n  .                   con <- file(fl <- tempfile(), \"w\")\n  .                   sink(con, type = \"message\")\n  .                   tryCatch({\n  .                     xx <- close(file(\"https://bioconductor.org\"))\n  .                   }, error = function(e) {\n  .                     message(conditionMessage(e))\n  .                   })\n  .                   sink(type = \"message\")\n  .                   close(con)\n  .                   if (!length(readLines(fl))) a[\"BioCsoft\", \"URL\"] <- sub(\"^http:\", \n  .                     \"https:\", a[\"BioCsoft\", \"URL\"])\n  .                 }\n  .                 if (vers >= \"3.5.0\") {\n  .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n  .                     \"3.7\", a[\"BioCsoft\", \"URL\"])\n  .                 } else if (vers >= \"3.4.0\") {\n  .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n  .                     \"3.6\", a[\"BioCsoft\", \"URL\"])\n  .                 } else if (vers >= \"3.3.0\") {\n  .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n  .                     \"3.4\", a[\"BioCsoft\", \"URL\"])\n  .                 } else if (vers >= \"3.2\") {\n  .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n  .                     \"3.2\", a[\"BioCsoft\", \"URL\"])\n  .                 } else if (vers == \"3.1.1\") {\n  .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n  .                     \"3.0\", a[\"BioCsoft\", \"URL\"])\n  .                 } else if (vers == \"3.1.0\") {\n  .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n  .                     \"2.14\", a[\"BioCsoft\", \"URL\"])\n  .                 } else if (vers >= \"2.15\" && vers < \"2.16\") {\n  .                   a[\"BioCsoft\", \"URL\"] <- sub(as.character(biocVers), \n  .                     \"2.11\", a[\"BioCsoft\", \"URL\"])\n  .                   biocVers <- numeric_version(\"2.11\")\n  .                 }\n  .                 install.packages(\"BiocInstaller\", repos = a[\"BioCsoft\", \n  .                   \"URL\"])\n  .                 if (!suppressWarnings(require(\"BiocInstaller\", \n  .                   quietly = TRUE))) {\n  .                   path0 <- \"//bioconductor.org/packages\"\n  .                   path <- sprintf(\"%s/%s/bioc\", path0, as.character(biocVers))\n  .                   txt0 <- \"'biocLite.R' failed to install 'BiocInstaller',\\n                        use 'install.packages(\\\"BiocInstaller\\\",\\n                        repos=\\\"https:%s\\\")' or\\n                        'install.packages(\\\"BiocInstaller\\\", repos=\\\"http:%s\\\")'\"\n  .                   txt <- sprintf(txt0, path, path)\n  .                   message(paste(strwrap(txt), collapse = \"\\n  \"))\n  .                 }\n  .             }\n  .         } else {\n  .             installerVersion <- utils::packageVersion(\"BiocInstaller\")\n  .             test0 <- (vers > \"3.1.2\") && !identical(getOption(\"pkgType\"), \n  .                 \"source\") && (installerVersion >= \"1.16.0\") && \n  .                 (installerVersion <= \"1.16.4\")\n  .             if (test0) {\n  .                 if (installerVersion < \"1.16.4\") {\n  .                   txt <- \"Update BiocInstaller with\\n                         oldPkgType=options(pkgType=\\\"source\\\");\\n                         biocLite(\\\"BiocInstaller\\\"); options(oldPkgType)\"\n  .                   message(paste(strwrap(txt, exdent = 2), collapse = \"\\n\"))\n  .                 }\n  .                 if (vers >= \"3.2\") {\n  .                   path <- \"//bioconductor.org/biocLite.R\"\n  .                   txt <- sprintf(\"BiocInstaller version %s is too old for\\n                         R version %s; remove.packages(\\\"BiocInstaller\\\"),\\n                         re-start R, then source(\\\"https:%s\\\") or\\n                         source(\\\"http:%s\\\")\", \n  .                     biocVers, vers, path, path)\n  .                   warning(paste(strwrap(txt, exdent = 2), collapse = \"\\n\"))\n  .                 }\n  .             }\n  .         }\n  .     } else {\n  .         tryCatch({\n  .             source(\"https://bioconductor.org/getBioC.R\")\n  .         }, error = function(e) {\n  .             warning(\"https: failed (\", conditionMessage(e), \"), trying http\", \n  .                 immediate. = TRUE)\n  .             source(\"http://bioconductor.org/getBioC.R\")\n  .         })\n  .         biocLite <<- function(pkgs, groupName = \"lite\", ...) {\n  .             if (missing(pkgs)) biocinstall(groupName = groupName, \n  .                 ...) else biocinstall(pkgs = pkgs, groupName = groupName, \n  .                 ...)\n  .         }\n  .     }\n  . }), new.env())",
      "11. stop(\"With R version 3.5 or greater, install Bioconductor \", \n  .     \"packages using BiocManager; see https://bioconductor.org/install\", \n  .     call. = FALSE)"
     ]
    }
   ],
   "source": [
    "rm(list=ls())\n",
    "\n",
    "library(tidyverse)\n",
    "library(dplyr)\n",
    "library(readr)\n",
    "library(stringr)\n",
    "library(ggplot2)\n",
    "library(qqman)\n",
    "\n",
    "\n",
    "\n",
    "#if (!require(``BiocManager'', quietly = TRUE)) install.packages(`BiocManager')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1ed22455-f2c7-44b1-9180-a80edd3f7fef",
   "metadata": {},
   "source": [
    "# Analyze Variant Effect Predictor: VEP results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "40fded79-c565-48f4-a726-2bac6dae682f",
   "metadata": {},
   "outputs": [],
   "source": [
    "resultVEP <- read.table(\"combined.jc.ss.gs.vep.out\", sep=\"\\t\",  header = FALSE, comment.char = \"#\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "a059f88a-ced0-4f64-854b-503dbff8ecfa",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A data.frame: 37815 × 14</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>V1</th><th scope=col>V2</th><th scope=col>V3</th><th scope=col>V4</th><th scope=col>V5</th><th scope=col>V6</th><th scope=col>V7</th><th scope=col>V8</th><th scope=col>V9</th><th scope=col>V10</th><th scope=col>V11</th><th scope=col>V12</th><th scope=col>V13</th><th scope=col>V14</th></tr>\n",
       "\t<tr><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000229768</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000330432</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;DISTANCE=3458;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000337049</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000360422</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;DISTANCE=3320;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000414028</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000419506</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000428397</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;DISTANCE=3567;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000434900</td><td>Transcript</td><td>intron_variant       </td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1                 </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000435918</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000452687</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;DISTANCE=3674;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000518759</td><td>Transcript</td><td>intron_variant       </td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1                 </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000519083</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000520282</td><td>Transcript</td><td>intron_variant       </td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;STRAND=1;FLAGS=cds_end_NF;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1</td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000520708</td><td>Transcript</td><td>intron_variant       </td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1                 </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000522739</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;DISTANCE=3693;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000523520</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;DISTANCE=3582;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000524150</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;DISTANCE=3697;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   </td></tr>\n",
       "\t<tr><td>chr6_154035782_T/C</td><td>chr6:154035782</td><td>C</td><td>ENSG00000112038</td><td>ENST00000524163</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs79996746,COSV57674301</td><td>IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   </td></tr>\n",
       "\t<tr><td>chr6_154036851_T/G</td><td>chr6:154036851</td><td>G</td><td>ENSG00000112038</td><td>ENST00000229768</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs12210856             </td><td>IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  </td></tr>\n",
       "\t<tr><td>chr6_154036851_T/G</td><td>chr6:154036851</td><td>G</td><td>ENSG00000112038</td><td>ENST00000330432</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs12210856             </td><td>IMPACT=MODIFIER;DISTANCE=2389;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  </td></tr>\n",
       "\t<tr><td>chr6_154036851_T/G</td><td>chr6:154036851</td><td>G</td><td>ENSG00000112038</td><td>ENST00000337049</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs12210856             </td><td>IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  </td></tr>\n",
       "\t<tr><td>chr6_154036851_T/G</td><td>chr6:154036851</td><td>G</td><td>ENSG00000112038</td><td>ENST00000360422</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs12210856             </td><td>IMPACT=MODIFIER;DISTANCE=2251;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  </td></tr>\n",
       "\t<tr><td>chr6_154036851_T/G</td><td>chr6:154036851</td><td>G</td><td>ENSG00000112038</td><td>ENST00000414028</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs12210856             </td><td>IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  </td></tr>\n",
       "\t<tr><td>chr6_154036851_T/G</td><td>chr6:154036851</td><td>G</td><td>ENSG00000112038</td><td>ENST00000419506</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs12210856             </td><td>IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  </td></tr>\n",
       "\t<tr><td>chr6_154036851_T/G</td><td>chr6:154036851</td><td>G</td><td>ENSG00000112038</td><td>ENST00000428397</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs12210856             </td><td>IMPACT=MODIFIER;DISTANCE=2498;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  </td></tr>\n",
       "\t<tr><td>chr6_154036851_T/G</td><td>chr6:154036851</td><td>G</td><td>ENSG00000112038</td><td>ENST00000434900</td><td>Transcript</td><td>intron_variant       </td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs12210856             </td><td>IMPACT=MODIFIER;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                                </td></tr>\n",
       "\t<tr><td>chr6_154036851_T/G</td><td>chr6:154036851</td><td>G</td><td>ENSG00000112038</td><td>ENST00000435918</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs12210856             </td><td>IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  </td></tr>\n",
       "\t<tr><td>chr6_154036851_T/G</td><td>chr6:154036851</td><td>G</td><td>ENSG00000112038</td><td>ENST00000452687</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs12210856             </td><td>IMPACT=MODIFIER;DISTANCE=2605;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  </td></tr>\n",
       "\t<tr><td>chr6_154036851_T/G</td><td>chr6:154036851</td><td>G</td><td>ENSG00000112038</td><td>ENST00000518759</td><td>Transcript</td><td>intron_variant       </td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs12210856             </td><td>IMPACT=MODIFIER;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                                </td></tr>\n",
       "\t<tr><td>chr6_154036851_T/G</td><td>chr6:154036851</td><td>G</td><td>ENSG00000112038</td><td>ENST00000519083</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs12210856             </td><td>IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  </td></tr>\n",
       "\t<tr><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td></tr>\n",
       "\t<tr><td>chr3_114181373_A/G  </td><td>chr3:114181373          </td><td>G</td><td>ENSG00000151577</td><td>ENST00000295881</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs9825563  </td><td>IMPACT=MODIFIER;DISTANCE=2321;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR         </td></tr>\n",
       "\t<tr><td>chr3_114181373_A/G  </td><td>chr3:114181373          </td><td>G</td><td>ENSG00000151577</td><td>ENST00000383673</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs9825563  </td><td>IMPACT=MODIFIER;DISTANCE=2321;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR         </td></tr>\n",
       "\t<tr><td>chr3_114181373_A/G  </td><td>chr3:114181373          </td><td>G</td><td>ENSG00000151577</td><td>ENST00000460779</td><td>Transcript</td><td>intron_variant       </td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs9825563  </td><td>IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR                       </td></tr>\n",
       "\t<tr><td>chr3_114181373_A/G  </td><td>chr3:114181373          </td><td>G</td><td>ENSG00000151577</td><td>ENST00000467632</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs9825563  </td><td>IMPACT=MODIFIER;DISTANCE=2675;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR         </td></tr>\n",
       "\t<tr><td>chr3_114181373_A/G  </td><td>chr3:114181373          </td><td>G</td><td>ENSG00000151577</td><td>ENST00000698213</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs9825563  </td><td>IMPACT=MODIFIER;DISTANCE=2321;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR         </td></tr>\n",
       "\t<tr><td>chr3_114181893_C/T  </td><td>chr3:114181893          </td><td>T</td><td>ENSG00000151577</td><td>ENST00000295881</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs75970639 </td><td>IMPACT=MODIFIER;DISTANCE=2841;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS         </td></tr>\n",
       "\t<tr><td>chr3_114181893_C/T  </td><td>chr3:114181893          </td><td>T</td><td>ENSG00000151577</td><td>ENST00000383673</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs75970639 </td><td>IMPACT=MODIFIER;DISTANCE=2841;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS         </td></tr>\n",
       "\t<tr><td>chr3_114181893_C/T  </td><td>chr3:114181893          </td><td>T</td><td>ENSG00000151577</td><td>ENST00000460779</td><td>Transcript</td><td>intron_variant       </td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs75970639 </td><td>IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS                       </td></tr>\n",
       "\t<tr><td>chr3_114181893_C/T  </td><td>chr3:114181893          </td><td>T</td><td>ENSG00000151577</td><td>ENST00000467632</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs75970639 </td><td>IMPACT=MODIFIER;DISTANCE=3195;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS         </td></tr>\n",
       "\t<tr><td>chr3_114181893_C/T  </td><td>chr3:114181893          </td><td>T</td><td>ENSG00000151577</td><td>ENST00000698213</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs75970639 </td><td>IMPACT=MODIFIER;DISTANCE=2841;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS         </td></tr>\n",
       "\t<tr><td>chr3_114182256_G/C  </td><td>chr3:114182256          </td><td>C</td><td>ENSG00000151577</td><td>ENST00000295881</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs62267154 </td><td>IMPACT=MODIFIER;DISTANCE=3204;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR </td></tr>\n",
       "\t<tr><td>chr3_114182256_G/C  </td><td>chr3:114182256          </td><td>C</td><td>ENSG00000151577</td><td>ENST00000383673</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs62267154 </td><td>IMPACT=MODIFIER;DISTANCE=3204;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR </td></tr>\n",
       "\t<tr><td>chr3_114182256_G/C  </td><td>chr3:114182256          </td><td>C</td><td>ENSG00000151577</td><td>ENST00000460779</td><td>Transcript</td><td>intron_variant       </td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs62267154 </td><td>IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR               </td></tr>\n",
       "\t<tr><td>chr3_114182256_G/C  </td><td>chr3:114182256          </td><td>C</td><td>ENSG00000151577</td><td>ENST00000467632</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs62267154 </td><td>IMPACT=MODIFIER;DISTANCE=3558;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR </td></tr>\n",
       "\t<tr><td>chr3_114182256_G/C  </td><td>chr3:114182256          </td><td>C</td><td>ENSG00000151577</td><td>ENST00000698213</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs62267154 </td><td>IMPACT=MODIFIER;DISTANCE=3204;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR </td></tr>\n",
       "\t<tr><td>chr3_114182326_TTA/-</td><td>chr3:114182326-114182328</td><td>-</td><td>ENSG00000151577</td><td>ENST00000295881</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs150772796</td><td>IMPACT=MODIFIER;DISTANCE=3274;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR         </td></tr>\n",
       "\t<tr><td>chr3_114182326_TTA/-</td><td>chr3:114182326-114182328</td><td>-</td><td>ENSG00000151577</td><td>ENST00000383673</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs150772796</td><td>IMPACT=MODIFIER;DISTANCE=3274;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR         </td></tr>\n",
       "\t<tr><td>chr3_114182326_TTA/-</td><td>chr3:114182326-114182328</td><td>-</td><td>ENSG00000151577</td><td>ENST00000460779</td><td>Transcript</td><td>intron_variant       </td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs150772796</td><td>IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR                       </td></tr>\n",
       "\t<tr><td>chr3_114182326_TTA/-</td><td>chr3:114182326-114182328</td><td>-</td><td>ENSG00000151577</td><td>ENST00000467632</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs150772796</td><td>IMPACT=MODIFIER;DISTANCE=3628;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR         </td></tr>\n",
       "\t<tr><td>chr3_114182326_TTA/-</td><td>chr3:114182326-114182328</td><td>-</td><td>ENSG00000151577</td><td>ENST00000698213</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs150772796</td><td>IMPACT=MODIFIER;DISTANCE=3274;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR         </td></tr>\n",
       "\t<tr><td>chr3_114182483_-/T  </td><td>chr3:114182482-114182483</td><td>T</td><td>ENSG00000151577</td><td>ENST00000295881</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs145438954</td><td>IMPACT=MODIFIER;DISTANCE=3430;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR         </td></tr>\n",
       "\t<tr><td>chr3_114182483_-/T  </td><td>chr3:114182482-114182483</td><td>T</td><td>ENSG00000151577</td><td>ENST00000383673</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs145438954</td><td>IMPACT=MODIFIER;DISTANCE=3430;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR         </td></tr>\n",
       "\t<tr><td>chr3_114182483_-/T  </td><td>chr3:114182482-114182483</td><td>T</td><td>ENSG00000151577</td><td>ENST00000460779</td><td>Transcript</td><td>intron_variant       </td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs145438954</td><td>IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR                       </td></tr>\n",
       "\t<tr><td>chr3_114182483_-/T  </td><td>chr3:114182482-114182483</td><td>T</td><td>ENSG00000151577</td><td>ENST00000467632</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs145438954</td><td>IMPACT=MODIFIER;DISTANCE=3784;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR         </td></tr>\n",
       "\t<tr><td>chr3_114182483_-/T  </td><td>chr3:114182482-114182483</td><td>T</td><td>ENSG00000151577</td><td>ENST00000698213</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs145438954</td><td>IMPACT=MODIFIER;DISTANCE=3430;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR         </td></tr>\n",
       "\t<tr><td>chr3_114182758_T/C  </td><td>chr3:114182758          </td><td>C</td><td>ENSG00000151577</td><td>ENST00000295881</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs112915369</td><td>IMPACT=MODIFIER;DISTANCE=3706;STRAND=-1;MAX_AF=0.08544;MAX_AF_POPS=gnomADg_MID</td></tr>\n",
       "\t<tr><td>chr3_114182758_T/C  </td><td>chr3:114182758          </td><td>C</td><td>ENSG00000151577</td><td>ENST00000383673</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs112915369</td><td>IMPACT=MODIFIER;DISTANCE=3706;STRAND=-1;MAX_AF=0.08544;MAX_AF_POPS=gnomADg_MID</td></tr>\n",
       "\t<tr><td>chr3_114182758_T/C  </td><td>chr3:114182758          </td><td>C</td><td>ENSG00000151577</td><td>ENST00000460779</td><td>Transcript</td><td>intron_variant       </td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs112915369</td><td>IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.08544;MAX_AF_POPS=gnomADg_MID              </td></tr>\n",
       "\t<tr><td>chr3_114182758_T/C  </td><td>chr3:114182758          </td><td>C</td><td>ENSG00000151577</td><td>ENST00000467632</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs112915369</td><td>IMPACT=MODIFIER;DISTANCE=4060;STRAND=-1;MAX_AF=0.08544;MAX_AF_POPS=gnomADg_MID</td></tr>\n",
       "\t<tr><td>chr3_114182758_T/C  </td><td>chr3:114182758          </td><td>C</td><td>ENSG00000151577</td><td>ENST00000698213</td><td>Transcript</td><td>upstream_gene_variant</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>rs112915369</td><td>IMPACT=MODIFIER;DI                                                            </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 37815 × 14\n",
       "\\begin{tabular}{llllllllllllll}\n",
       " V1 & V2 & V3 & V4 & V5 & V6 & V7 & V8 & V9 & V10 & V11 & V12 & V13 & V14\\\\\n",
       " <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr>\\\\\n",
       "\\hline\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000229768 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1   \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000330432 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;DISTANCE=3458;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1   \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000337049 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1   \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000360422 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;DISTANCE=3320;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1   \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000414028 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1   \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000419506 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1   \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000428397 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;DISTANCE=3567;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1   \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000434900 & Transcript & intron\\_variant        & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1                 \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000435918 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1   \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000452687 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;DISTANCE=3674;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1   \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000518759 & Transcript & intron\\_variant        & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1                 \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000519083 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1   \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000520282 & Transcript & intron\\_variant        & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;STRAND=1;FLAGS=cds\\_end\\_NF;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1\\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000520708 & Transcript & intron\\_variant        & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1                 \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000522739 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;DISTANCE=3693;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1   \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000523520 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;DISTANCE=3582;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1   \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000524150 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;DISTANCE=3697;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1   \\\\\n",
       "\t chr6\\_154035782\\_T/C & chr6:154035782 & C & ENSG00000112038 & ENST00000524163 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs79996746,COSV57674301 & IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX\\_AF=0.07796;MAX\\_AF\\_POPS=gnomADg\\_SAS;SOMATIC=0,1;PHENO=0,1   \\\\\n",
       "\t chr6\\_154036851\\_T/G & chr6:154036851 & G & ENSG00000112038 & ENST00000229768 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs12210856              & IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX\\_AF=0.3517;MAX\\_AF\\_POPS=AFR                                  \\\\\n",
       "\t chr6\\_154036851\\_T/G & chr6:154036851 & G & ENSG00000112038 & ENST00000330432 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs12210856              & IMPACT=MODIFIER;DISTANCE=2389;STRAND=1;MAX\\_AF=0.3517;MAX\\_AF\\_POPS=AFR                                  \\\\\n",
       "\t chr6\\_154036851\\_T/G & chr6:154036851 & G & ENSG00000112038 & ENST00000337049 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs12210856              & IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX\\_AF=0.3517;MAX\\_AF\\_POPS=AFR                                  \\\\\n",
       "\t chr6\\_154036851\\_T/G & chr6:154036851 & G & ENSG00000112038 & ENST00000360422 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs12210856              & IMPACT=MODIFIER;DISTANCE=2251;STRAND=1;MAX\\_AF=0.3517;MAX\\_AF\\_POPS=AFR                                  \\\\\n",
       "\t chr6\\_154036851\\_T/G & chr6:154036851 & G & ENSG00000112038 & ENST00000414028 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs12210856              & IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX\\_AF=0.3517;MAX\\_AF\\_POPS=AFR                                  \\\\\n",
       "\t chr6\\_154036851\\_T/G & chr6:154036851 & G & ENSG00000112038 & ENST00000419506 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs12210856              & IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX\\_AF=0.3517;MAX\\_AF\\_POPS=AFR                                  \\\\\n",
       "\t chr6\\_154036851\\_T/G & chr6:154036851 & G & ENSG00000112038 & ENST00000428397 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs12210856              & IMPACT=MODIFIER;DISTANCE=2498;STRAND=1;MAX\\_AF=0.3517;MAX\\_AF\\_POPS=AFR                                  \\\\\n",
       "\t chr6\\_154036851\\_T/G & chr6:154036851 & G & ENSG00000112038 & ENST00000434900 & Transcript & intron\\_variant        & - & - & - & - & - & rs12210856              & IMPACT=MODIFIER;STRAND=1;MAX\\_AF=0.3517;MAX\\_AF\\_POPS=AFR                                                \\\\\n",
       "\t chr6\\_154036851\\_T/G & chr6:154036851 & G & ENSG00000112038 & ENST00000435918 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs12210856              & IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX\\_AF=0.3517;MAX\\_AF\\_POPS=AFR                                  \\\\\n",
       "\t chr6\\_154036851\\_T/G & chr6:154036851 & G & ENSG00000112038 & ENST00000452687 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs12210856              & IMPACT=MODIFIER;DISTANCE=2605;STRAND=1;MAX\\_AF=0.3517;MAX\\_AF\\_POPS=AFR                                  \\\\\n",
       "\t chr6\\_154036851\\_T/G & chr6:154036851 & G & ENSG00000112038 & ENST00000518759 & Transcript & intron\\_variant        & - & - & - & - & - & rs12210856              & IMPACT=MODIFIER;STRAND=1;MAX\\_AF=0.3517;MAX\\_AF\\_POPS=AFR                                                \\\\\n",
       "\t chr6\\_154036851\\_T/G & chr6:154036851 & G & ENSG00000112038 & ENST00000519083 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs12210856              & IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX\\_AF=0.3517;MAX\\_AF\\_POPS=AFR                                  \\\\\n",
       "\t ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮\\\\\n",
       "\t chr3\\_114181373\\_A/G   & chr3:114181373           & G & ENSG00000151577 & ENST00000295881 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs9825563   & IMPACT=MODIFIER;DISTANCE=2321;STRAND=-1;MAX\\_AF=0.5204;MAX\\_AF\\_POPS=AFR         \\\\\n",
       "\t chr3\\_114181373\\_A/G   & chr3:114181373           & G & ENSG00000151577 & ENST00000383673 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs9825563   & IMPACT=MODIFIER;DISTANCE=2321;STRAND=-1;MAX\\_AF=0.5204;MAX\\_AF\\_POPS=AFR         \\\\\n",
       "\t chr3\\_114181373\\_A/G   & chr3:114181373           & G & ENSG00000151577 & ENST00000460779 & Transcript & intron\\_variant        & - & - & - & - & - & rs9825563   & IMPACT=MODIFIER;STRAND=-1;MAX\\_AF=0.5204;MAX\\_AF\\_POPS=AFR                       \\\\\n",
       "\t chr3\\_114181373\\_A/G   & chr3:114181373           & G & ENSG00000151577 & ENST00000467632 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs9825563   & IMPACT=MODIFIER;DISTANCE=2675;STRAND=-1;MAX\\_AF=0.5204;MAX\\_AF\\_POPS=AFR         \\\\\n",
       "\t chr3\\_114181373\\_A/G   & chr3:114181373           & G & ENSG00000151577 & ENST00000698213 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs9825563   & IMPACT=MODIFIER;DISTANCE=2321;STRAND=-1;MAX\\_AF=0.5204;MAX\\_AF\\_POPS=AFR         \\\\\n",
       "\t chr3\\_114181893\\_C/T   & chr3:114181893           & T & ENSG00000151577 & ENST00000295881 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs75970639  & IMPACT=MODIFIER;DISTANCE=2841;STRAND=-1;MAX\\_AF=0.1513;MAX\\_AF\\_POPS=SAS         \\\\\n",
       "\t chr3\\_114181893\\_C/T   & chr3:114181893           & T & ENSG00000151577 & ENST00000383673 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs75970639  & IMPACT=MODIFIER;DISTANCE=2841;STRAND=-1;MAX\\_AF=0.1513;MAX\\_AF\\_POPS=SAS         \\\\\n",
       "\t chr3\\_114181893\\_C/T   & chr3:114181893           & T & ENSG00000151577 & ENST00000460779 & Transcript & intron\\_variant        & - & - & - & - & - & rs75970639  & IMPACT=MODIFIER;STRAND=-1;MAX\\_AF=0.1513;MAX\\_AF\\_POPS=SAS                       \\\\\n",
       "\t chr3\\_114181893\\_C/T   & chr3:114181893           & T & ENSG00000151577 & ENST00000467632 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs75970639  & IMPACT=MODIFIER;DISTANCE=3195;STRAND=-1;MAX\\_AF=0.1513;MAX\\_AF\\_POPS=SAS         \\\\\n",
       "\t chr3\\_114181893\\_C/T   & chr3:114181893           & T & ENSG00000151577 & ENST00000698213 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs75970639  & IMPACT=MODIFIER;DISTANCE=2841;STRAND=-1;MAX\\_AF=0.1513;MAX\\_AF\\_POPS=SAS         \\\\\n",
       "\t chr3\\_114182256\\_G/C   & chr3:114182256           & C & ENSG00000151577 & ENST00000295881 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs62267154  & IMPACT=MODIFIER;DISTANCE=3204;STRAND=-1;MAX\\_AF=0.2331;MAX\\_AF\\_POPS=gnomADg\\_AMR \\\\\n",
       "\t chr3\\_114182256\\_G/C   & chr3:114182256           & C & ENSG00000151577 & ENST00000383673 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs62267154  & IMPACT=MODIFIER;DISTANCE=3204;STRAND=-1;MAX\\_AF=0.2331;MAX\\_AF\\_POPS=gnomADg\\_AMR \\\\\n",
       "\t chr3\\_114182256\\_G/C   & chr3:114182256           & C & ENSG00000151577 & ENST00000460779 & Transcript & intron\\_variant        & - & - & - & - & - & rs62267154  & IMPACT=MODIFIER;STRAND=-1;MAX\\_AF=0.2331;MAX\\_AF\\_POPS=gnomADg\\_AMR               \\\\\n",
       "\t chr3\\_114182256\\_G/C   & chr3:114182256           & C & ENSG00000151577 & ENST00000467632 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs62267154  & IMPACT=MODIFIER;DISTANCE=3558;STRAND=-1;MAX\\_AF=0.2331;MAX\\_AF\\_POPS=gnomADg\\_AMR \\\\\n",
       "\t chr3\\_114182256\\_G/C   & chr3:114182256           & C & ENSG00000151577 & ENST00000698213 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs62267154  & IMPACT=MODIFIER;DISTANCE=3204;STRAND=-1;MAX\\_AF=0.2331;MAX\\_AF\\_POPS=gnomADg\\_AMR \\\\\n",
       "\t chr3\\_114182326\\_TTA/- & chr3:114182326-114182328 & - & ENSG00000151577 & ENST00000295881 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs150772796 & IMPACT=MODIFIER;DISTANCE=3274;STRAND=-1;MAX\\_AF=0.2284;MAX\\_AF\\_POPS=AFR         \\\\\n",
       "\t chr3\\_114182326\\_TTA/- & chr3:114182326-114182328 & - & ENSG00000151577 & ENST00000383673 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs150772796 & IMPACT=MODIFIER;DISTANCE=3274;STRAND=-1;MAX\\_AF=0.2284;MAX\\_AF\\_POPS=AFR         \\\\\n",
       "\t chr3\\_114182326\\_TTA/- & chr3:114182326-114182328 & - & ENSG00000151577 & ENST00000460779 & Transcript & intron\\_variant        & - & - & - & - & - & rs150772796 & IMPACT=MODIFIER;STRAND=-1;MAX\\_AF=0.2284;MAX\\_AF\\_POPS=AFR                       \\\\\n",
       "\t chr3\\_114182326\\_TTA/- & chr3:114182326-114182328 & - & ENSG00000151577 & ENST00000467632 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs150772796 & IMPACT=MODIFIER;DISTANCE=3628;STRAND=-1;MAX\\_AF=0.2284;MAX\\_AF\\_POPS=AFR         \\\\\n",
       "\t chr3\\_114182326\\_TTA/- & chr3:114182326-114182328 & - & ENSG00000151577 & ENST00000698213 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs150772796 & IMPACT=MODIFIER;DISTANCE=3274;STRAND=-1;MAX\\_AF=0.2284;MAX\\_AF\\_POPS=AFR         \\\\\n",
       "\t chr3\\_114182483\\_-/T   & chr3:114182482-114182483 & T & ENSG00000151577 & ENST00000295881 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs145438954 & IMPACT=MODIFIER;DISTANCE=3430;STRAND=-1;MAX\\_AF=0.1415;MAX\\_AF\\_POPS=AFR         \\\\\n",
       "\t chr3\\_114182483\\_-/T   & chr3:114182482-114182483 & T & ENSG00000151577 & ENST00000383673 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs145438954 & IMPACT=MODIFIER;DISTANCE=3430;STRAND=-1;MAX\\_AF=0.1415;MAX\\_AF\\_POPS=AFR         \\\\\n",
       "\t chr3\\_114182483\\_-/T   & chr3:114182482-114182483 & T & ENSG00000151577 & ENST00000460779 & Transcript & intron\\_variant        & - & - & - & - & - & rs145438954 & IMPACT=MODIFIER;STRAND=-1;MAX\\_AF=0.1415;MAX\\_AF\\_POPS=AFR                       \\\\\n",
       "\t chr3\\_114182483\\_-/T   & chr3:114182482-114182483 & T & ENSG00000151577 & ENST00000467632 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs145438954 & IMPACT=MODIFIER;DISTANCE=3784;STRAND=-1;MAX\\_AF=0.1415;MAX\\_AF\\_POPS=AFR         \\\\\n",
       "\t chr3\\_114182483\\_-/T   & chr3:114182482-114182483 & T & ENSG00000151577 & ENST00000698213 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs145438954 & IMPACT=MODIFIER;DISTANCE=3430;STRAND=-1;MAX\\_AF=0.1415;MAX\\_AF\\_POPS=AFR         \\\\\n",
       "\t chr3\\_114182758\\_T/C   & chr3:114182758           & C & ENSG00000151577 & ENST00000295881 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs112915369 & IMPACT=MODIFIER;DISTANCE=3706;STRAND=-1;MAX\\_AF=0.08544;MAX\\_AF\\_POPS=gnomADg\\_MID\\\\\n",
       "\t chr3\\_114182758\\_T/C   & chr3:114182758           & C & ENSG00000151577 & ENST00000383673 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs112915369 & IMPACT=MODIFIER;DISTANCE=3706;STRAND=-1;MAX\\_AF=0.08544;MAX\\_AF\\_POPS=gnomADg\\_MID\\\\\n",
       "\t chr3\\_114182758\\_T/C   & chr3:114182758           & C & ENSG00000151577 & ENST00000460779 & Transcript & intron\\_variant        & - & - & - & - & - & rs112915369 & IMPACT=MODIFIER;STRAND=-1;MAX\\_AF=0.08544;MAX\\_AF\\_POPS=gnomADg\\_MID              \\\\\n",
       "\t chr3\\_114182758\\_T/C   & chr3:114182758           & C & ENSG00000151577 & ENST00000467632 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs112915369 & IMPACT=MODIFIER;DISTANCE=4060;STRAND=-1;MAX\\_AF=0.08544;MAX\\_AF\\_POPS=gnomADg\\_MID\\\\\n",
       "\t chr3\\_114182758\\_T/C   & chr3:114182758           & C & ENSG00000151577 & ENST00000698213 & Transcript & upstream\\_gene\\_variant & - & - & - & - & - & rs112915369 & IMPACT=MODIFIER;DI                                                            \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 37815 × 14\n",
       "\n",
       "| V1 &lt;chr&gt; | V2 &lt;chr&gt; | V3 &lt;chr&gt; | V4 &lt;chr&gt; | V5 &lt;chr&gt; | V6 &lt;chr&gt; | V7 &lt;chr&gt; | V8 &lt;chr&gt; | V9 &lt;chr&gt; | V10 &lt;chr&gt; | V11 &lt;chr&gt; | V12 &lt;chr&gt; | V13 &lt;chr&gt; | V14 &lt;chr&gt; |\n",
       "|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000229768 | Transcript | upstream_gene_variant | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1    |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000330432 | Transcript | upstream_gene_variant | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;DISTANCE=3458;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1    |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000337049 | Transcript | upstream_gene_variant | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1    |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000360422 | Transcript | upstream_gene_variant | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;DISTANCE=3320;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1    |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000414028 | Transcript | upstream_gene_variant | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1    |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000419506 | Transcript | upstream_gene_variant | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1    |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000428397 | Transcript | upstream_gene_variant | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;DISTANCE=3567;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1    |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000434900 | Transcript | intron_variant        | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1                  |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000435918 | Transcript | upstream_gene_variant | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1    |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000452687 | Transcript | upstream_gene_variant | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;DISTANCE=3674;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1    |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000518759 | Transcript | intron_variant        | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1                  |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000519083 | Transcript | upstream_gene_variant | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1    |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000520282 | Transcript | intron_variant        | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;STRAND=1;FLAGS=cds_end_NF;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1 |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000520708 | Transcript | intron_variant        | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1                  |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000522739 | Transcript | upstream_gene_variant | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;DISTANCE=3693;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1    |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000523520 | Transcript | upstream_gene_variant | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;DISTANCE=3582;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1    |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000524150 | Transcript | upstream_gene_variant | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;DISTANCE=3697;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1    |\n",
       "| chr6_154035782_T/C | chr6:154035782 | C | ENSG00000112038 | ENST00000524163 | Transcript | upstream_gene_variant | - | - | - | - | - | rs79996746,COSV57674301 | IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1    |\n",
       "| chr6_154036851_T/G | chr6:154036851 | G | ENSG00000112038 | ENST00000229768 | Transcript | upstream_gene_variant | - | - | - | - | - | rs12210856              | IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                   |\n",
       "| chr6_154036851_T/G | chr6:154036851 | G | ENSG00000112038 | ENST00000330432 | Transcript | upstream_gene_variant | - | - | - | - | - | rs12210856              | IMPACT=MODIFIER;DISTANCE=2389;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                   |\n",
       "| chr6_154036851_T/G | chr6:154036851 | G | ENSG00000112038 | ENST00000337049 | Transcript | upstream_gene_variant | - | - | - | - | - | rs12210856              | IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                   |\n",
       "| chr6_154036851_T/G | chr6:154036851 | G | ENSG00000112038 | ENST00000360422 | Transcript | upstream_gene_variant | - | - | - | - | - | rs12210856              | IMPACT=MODIFIER;DISTANCE=2251;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                   |\n",
       "| chr6_154036851_T/G | chr6:154036851 | G | ENSG00000112038 | ENST00000414028 | Transcript | upstream_gene_variant | - | - | - | - | - | rs12210856              | IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                   |\n",
       "| chr6_154036851_T/G | chr6:154036851 | G | ENSG00000112038 | ENST00000419506 | Transcript | upstream_gene_variant | - | - | - | - | - | rs12210856              | IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                   |\n",
       "| chr6_154036851_T/G | chr6:154036851 | G | ENSG00000112038 | ENST00000428397 | Transcript | upstream_gene_variant | - | - | - | - | - | rs12210856              | IMPACT=MODIFIER;DISTANCE=2498;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                   |\n",
       "| chr6_154036851_T/G | chr6:154036851 | G | ENSG00000112038 | ENST00000434900 | Transcript | intron_variant        | - | - | - | - | - | rs12210856              | IMPACT=MODIFIER;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                                 |\n",
       "| chr6_154036851_T/G | chr6:154036851 | G | ENSG00000112038 | ENST00000435918 | Transcript | upstream_gene_variant | - | - | - | - | - | rs12210856              | IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                   |\n",
       "| chr6_154036851_T/G | chr6:154036851 | G | ENSG00000112038 | ENST00000452687 | Transcript | upstream_gene_variant | - | - | - | - | - | rs12210856              | IMPACT=MODIFIER;DISTANCE=2605;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                   |\n",
       "| chr6_154036851_T/G | chr6:154036851 | G | ENSG00000112038 | ENST00000518759 | Transcript | intron_variant        | - | - | - | - | - | rs12210856              | IMPACT=MODIFIER;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                                 |\n",
       "| chr6_154036851_T/G | chr6:154036851 | G | ENSG00000112038 | ENST00000519083 | Transcript | upstream_gene_variant | - | - | - | - | - | rs12210856              | IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                   |\n",
       "| ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ |\n",
       "| chr3_114181373_A/G   | chr3:114181373           | G | ENSG00000151577 | ENST00000295881 | Transcript | upstream_gene_variant | - | - | - | - | - | rs9825563   | IMPACT=MODIFIER;DISTANCE=2321;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR          |\n",
       "| chr3_114181373_A/G   | chr3:114181373           | G | ENSG00000151577 | ENST00000383673 | Transcript | upstream_gene_variant | - | - | - | - | - | rs9825563   | IMPACT=MODIFIER;DISTANCE=2321;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR          |\n",
       "| chr3_114181373_A/G   | chr3:114181373           | G | ENSG00000151577 | ENST00000460779 | Transcript | intron_variant        | - | - | - | - | - | rs9825563   | IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR                        |\n",
       "| chr3_114181373_A/G   | chr3:114181373           | G | ENSG00000151577 | ENST00000467632 | Transcript | upstream_gene_variant | - | - | - | - | - | rs9825563   | IMPACT=MODIFIER;DISTANCE=2675;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR          |\n",
       "| chr3_114181373_A/G   | chr3:114181373           | G | ENSG00000151577 | ENST00000698213 | Transcript | upstream_gene_variant | - | - | - | - | - | rs9825563   | IMPACT=MODIFIER;DISTANCE=2321;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR          |\n",
       "| chr3_114181893_C/T   | chr3:114181893           | T | ENSG00000151577 | ENST00000295881 | Transcript | upstream_gene_variant | - | - | - | - | - | rs75970639  | IMPACT=MODIFIER;DISTANCE=2841;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS          |\n",
       "| chr3_114181893_C/T   | chr3:114181893           | T | ENSG00000151577 | ENST00000383673 | Transcript | upstream_gene_variant | - | - | - | - | - | rs75970639  | IMPACT=MODIFIER;DISTANCE=2841;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS          |\n",
       "| chr3_114181893_C/T   | chr3:114181893           | T | ENSG00000151577 | ENST00000460779 | Transcript | intron_variant        | - | - | - | - | - | rs75970639  | IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS                        |\n",
       "| chr3_114181893_C/T   | chr3:114181893           | T | ENSG00000151577 | ENST00000467632 | Transcript | upstream_gene_variant | - | - | - | - | - | rs75970639  | IMPACT=MODIFIER;DISTANCE=3195;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS          |\n",
       "| chr3_114181893_C/T   | chr3:114181893           | T | ENSG00000151577 | ENST00000698213 | Transcript | upstream_gene_variant | - | - | - | - | - | rs75970639  | IMPACT=MODIFIER;DISTANCE=2841;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS          |\n",
       "| chr3_114182256_G/C   | chr3:114182256           | C | ENSG00000151577 | ENST00000295881 | Transcript | upstream_gene_variant | - | - | - | - | - | rs62267154  | IMPACT=MODIFIER;DISTANCE=3204;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR  |\n",
       "| chr3_114182256_G/C   | chr3:114182256           | C | ENSG00000151577 | ENST00000383673 | Transcript | upstream_gene_variant | - | - | - | - | - | rs62267154  | IMPACT=MODIFIER;DISTANCE=3204;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR  |\n",
       "| chr3_114182256_G/C   | chr3:114182256           | C | ENSG00000151577 | ENST00000460779 | Transcript | intron_variant        | - | - | - | - | - | rs62267154  | IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR                |\n",
       "| chr3_114182256_G/C   | chr3:114182256           | C | ENSG00000151577 | ENST00000467632 | Transcript | upstream_gene_variant | - | - | - | - | - | rs62267154  | IMPACT=MODIFIER;DISTANCE=3558;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR  |\n",
       "| chr3_114182256_G/C   | chr3:114182256           | C | ENSG00000151577 | ENST00000698213 | Transcript | upstream_gene_variant | - | - | - | - | - | rs62267154  | IMPACT=MODIFIER;DISTANCE=3204;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR  |\n",
       "| chr3_114182326_TTA/- | chr3:114182326-114182328 | - | ENSG00000151577 | ENST00000295881 | Transcript | upstream_gene_variant | - | - | - | - | - | rs150772796 | IMPACT=MODIFIER;DISTANCE=3274;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR          |\n",
       "| chr3_114182326_TTA/- | chr3:114182326-114182328 | - | ENSG00000151577 | ENST00000383673 | Transcript | upstream_gene_variant | - | - | - | - | - | rs150772796 | IMPACT=MODIFIER;DISTANCE=3274;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR          |\n",
       "| chr3_114182326_TTA/- | chr3:114182326-114182328 | - | ENSG00000151577 | ENST00000460779 | Transcript | intron_variant        | - | - | - | - | - | rs150772796 | IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR                        |\n",
       "| chr3_114182326_TTA/- | chr3:114182326-114182328 | - | ENSG00000151577 | ENST00000467632 | Transcript | upstream_gene_variant | - | - | - | - | - | rs150772796 | IMPACT=MODIFIER;DISTANCE=3628;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR          |\n",
       "| chr3_114182326_TTA/- | chr3:114182326-114182328 | - | ENSG00000151577 | ENST00000698213 | Transcript | upstream_gene_variant | - | - | - | - | - | rs150772796 | IMPACT=MODIFIER;DISTANCE=3274;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR          |\n",
       "| chr3_114182483_-/T   | chr3:114182482-114182483 | T | ENSG00000151577 | ENST00000295881 | Transcript | upstream_gene_variant | - | - | - | - | - | rs145438954 | IMPACT=MODIFIER;DISTANCE=3430;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR          |\n",
       "| chr3_114182483_-/T   | chr3:114182482-114182483 | T | ENSG00000151577 | ENST00000383673 | Transcript | upstream_gene_variant | - | - | - | - | - | rs145438954 | IMPACT=MODIFIER;DISTANCE=3430;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR          |\n",
       "| chr3_114182483_-/T   | chr3:114182482-114182483 | T | ENSG00000151577 | ENST00000460779 | Transcript | intron_variant        | - | - | - | - | - | rs145438954 | IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR                        |\n",
       "| chr3_114182483_-/T   | chr3:114182482-114182483 | T | ENSG00000151577 | ENST00000467632 | Transcript | upstream_gene_variant | - | - | - | - | - | rs145438954 | IMPACT=MODIFIER;DISTANCE=3784;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR          |\n",
       "| chr3_114182483_-/T   | chr3:114182482-114182483 | T | ENSG00000151577 | ENST00000698213 | Transcript | upstream_gene_variant | - | - | - | - | - | rs145438954 | IMPACT=MODIFIER;DISTANCE=3430;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR          |\n",
       "| chr3_114182758_T/C   | chr3:114182758           | C | ENSG00000151577 | ENST00000295881 | Transcript | upstream_gene_variant | - | - | - | - | - | rs112915369 | IMPACT=MODIFIER;DISTANCE=3706;STRAND=-1;MAX_AF=0.08544;MAX_AF_POPS=gnomADg_MID |\n",
       "| chr3_114182758_T/C   | chr3:114182758           | C | ENSG00000151577 | ENST00000383673 | Transcript | upstream_gene_variant | - | - | - | - | - | rs112915369 | IMPACT=MODIFIER;DISTANCE=3706;STRAND=-1;MAX_AF=0.08544;MAX_AF_POPS=gnomADg_MID |\n",
       "| chr3_114182758_T/C   | chr3:114182758           | C | ENSG00000151577 | ENST00000460779 | Transcript | intron_variant        | - | - | - | - | - | rs112915369 | IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.08544;MAX_AF_POPS=gnomADg_MID               |\n",
       "| chr3_114182758_T/C   | chr3:114182758           | C | ENSG00000151577 | ENST00000467632 | Transcript | upstream_gene_variant | - | - | - | - | - | rs112915369 | IMPACT=MODIFIER;DISTANCE=4060;STRAND=-1;MAX_AF=0.08544;MAX_AF_POPS=gnomADg_MID |\n",
       "| chr3_114182758_T/C   | chr3:114182758           | C | ENSG00000151577 | ENST00000698213 | Transcript | upstream_gene_variant | - | - | - | - | - | rs112915369 | IMPACT=MODIFIER;DI                                                             |\n",
       "\n"
      ],
      "text/plain": [
       "      V1                   V2                       V3 V4             \n",
       "1     chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "2     chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "3     chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "4     chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "5     chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "6     chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "7     chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "8     chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "9     chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "10    chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "11    chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "12    chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "13    chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "14    chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "15    chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "16    chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "17    chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "18    chr6_154035782_T/C   chr6:154035782           C  ENSG00000112038\n",
       "19    chr6_154036851_T/G   chr6:154036851           G  ENSG00000112038\n",
       "20    chr6_154036851_T/G   chr6:154036851           G  ENSG00000112038\n",
       "21    chr6_154036851_T/G   chr6:154036851           G  ENSG00000112038\n",
       "22    chr6_154036851_T/G   chr6:154036851           G  ENSG00000112038\n",
       "23    chr6_154036851_T/G   chr6:154036851           G  ENSG00000112038\n",
       "24    chr6_154036851_T/G   chr6:154036851           G  ENSG00000112038\n",
       "25    chr6_154036851_T/G   chr6:154036851           G  ENSG00000112038\n",
       "26    chr6_154036851_T/G   chr6:154036851           G  ENSG00000112038\n",
       "27    chr6_154036851_T/G   chr6:154036851           G  ENSG00000112038\n",
       "28    chr6_154036851_T/G   chr6:154036851           G  ENSG00000112038\n",
       "29    chr6_154036851_T/G   chr6:154036851           G  ENSG00000112038\n",
       "30    chr6_154036851_T/G   chr6:154036851           G  ENSG00000112038\n",
       "⋮     ⋮                    ⋮                        ⋮  ⋮              \n",
       "37786 chr3_114181373_A/G   chr3:114181373           G  ENSG00000151577\n",
       "37787 chr3_114181373_A/G   chr3:114181373           G  ENSG00000151577\n",
       "37788 chr3_114181373_A/G   chr3:114181373           G  ENSG00000151577\n",
       "37789 chr3_114181373_A/G   chr3:114181373           G  ENSG00000151577\n",
       "37790 chr3_114181373_A/G   chr3:114181373           G  ENSG00000151577\n",
       "37791 chr3_114181893_C/T   chr3:114181893           T  ENSG00000151577\n",
       "37792 chr3_114181893_C/T   chr3:114181893           T  ENSG00000151577\n",
       "37793 chr3_114181893_C/T   chr3:114181893           T  ENSG00000151577\n",
       "37794 chr3_114181893_C/T   chr3:114181893           T  ENSG00000151577\n",
       "37795 chr3_114181893_C/T   chr3:114181893           T  ENSG00000151577\n",
       "37796 chr3_114182256_G/C   chr3:114182256           C  ENSG00000151577\n",
       "37797 chr3_114182256_G/C   chr3:114182256           C  ENSG00000151577\n",
       "37798 chr3_114182256_G/C   chr3:114182256           C  ENSG00000151577\n",
       "37799 chr3_114182256_G/C   chr3:114182256           C  ENSG00000151577\n",
       "37800 chr3_114182256_G/C   chr3:114182256           C  ENSG00000151577\n",
       "37801 chr3_114182326_TTA/- chr3:114182326-114182328 -  ENSG00000151577\n",
       "37802 chr3_114182326_TTA/- chr3:114182326-114182328 -  ENSG00000151577\n",
       "37803 chr3_114182326_TTA/- chr3:114182326-114182328 -  ENSG00000151577\n",
       "37804 chr3_114182326_TTA/- chr3:114182326-114182328 -  ENSG00000151577\n",
       "37805 chr3_114182326_TTA/- chr3:114182326-114182328 -  ENSG00000151577\n",
       "37806 chr3_114182483_-/T   chr3:114182482-114182483 T  ENSG00000151577\n",
       "37807 chr3_114182483_-/T   chr3:114182482-114182483 T  ENSG00000151577\n",
       "37808 chr3_114182483_-/T   chr3:114182482-114182483 T  ENSG00000151577\n",
       "37809 chr3_114182483_-/T   chr3:114182482-114182483 T  ENSG00000151577\n",
       "37810 chr3_114182483_-/T   chr3:114182482-114182483 T  ENSG00000151577\n",
       "37811 chr3_114182758_T/C   chr3:114182758           C  ENSG00000151577\n",
       "37812 chr3_114182758_T/C   chr3:114182758           C  ENSG00000151577\n",
       "37813 chr3_114182758_T/C   chr3:114182758           C  ENSG00000151577\n",
       "37814 chr3_114182758_T/C   chr3:114182758           C  ENSG00000151577\n",
       "37815 chr3_114182758_T/C   chr3:114182758           C  ENSG00000151577\n",
       "      V5              V6         V7                    V8 V9 V10 V11 V12\n",
       "1     ENST00000229768 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "2     ENST00000330432 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "3     ENST00000337049 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "4     ENST00000360422 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "5     ENST00000414028 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "6     ENST00000419506 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "7     ENST00000428397 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "8     ENST00000434900 Transcript intron_variant        -  -  -   -   -  \n",
       "9     ENST00000435918 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "10    ENST00000452687 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "11    ENST00000518759 Transcript intron_variant        -  -  -   -   -  \n",
       "12    ENST00000519083 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "13    ENST00000520282 Transcript intron_variant        -  -  -   -   -  \n",
       "14    ENST00000520708 Transcript intron_variant        -  -  -   -   -  \n",
       "15    ENST00000522739 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "16    ENST00000523520 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "17    ENST00000524150 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "18    ENST00000524163 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "19    ENST00000229768 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "20    ENST00000330432 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "21    ENST00000337049 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "22    ENST00000360422 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "23    ENST00000414028 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "24    ENST00000419506 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "25    ENST00000428397 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "26    ENST00000434900 Transcript intron_variant        -  -  -   -   -  \n",
       "27    ENST00000435918 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "28    ENST00000452687 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "29    ENST00000518759 Transcript intron_variant        -  -  -   -   -  \n",
       "30    ENST00000519083 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "⋮     ⋮               ⋮          ⋮                     ⋮  ⋮  ⋮   ⋮   ⋮  \n",
       "37786 ENST00000295881 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37787 ENST00000383673 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37788 ENST00000460779 Transcript intron_variant        -  -  -   -   -  \n",
       "37789 ENST00000467632 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37790 ENST00000698213 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37791 ENST00000295881 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37792 ENST00000383673 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37793 ENST00000460779 Transcript intron_variant        -  -  -   -   -  \n",
       "37794 ENST00000467632 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37795 ENST00000698213 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37796 ENST00000295881 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37797 ENST00000383673 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37798 ENST00000460779 Transcript intron_variant        -  -  -   -   -  \n",
       "37799 ENST00000467632 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37800 ENST00000698213 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37801 ENST00000295881 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37802 ENST00000383673 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37803 ENST00000460779 Transcript intron_variant        -  -  -   -   -  \n",
       "37804 ENST00000467632 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37805 ENST00000698213 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37806 ENST00000295881 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37807 ENST00000383673 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37808 ENST00000460779 Transcript intron_variant        -  -  -   -   -  \n",
       "37809 ENST00000467632 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37810 ENST00000698213 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37811 ENST00000295881 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37812 ENST00000383673 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37813 ENST00000460779 Transcript intron_variant        -  -  -   -   -  \n",
       "37814 ENST00000467632 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "37815 ENST00000698213 Transcript upstream_gene_variant -  -  -   -   -  \n",
       "      V13                    \n",
       "1     rs79996746,COSV57674301\n",
       "2     rs79996746,COSV57674301\n",
       "3     rs79996746,COSV57674301\n",
       "4     rs79996746,COSV57674301\n",
       "5     rs79996746,COSV57674301\n",
       "6     rs79996746,COSV57674301\n",
       "7     rs79996746,COSV57674301\n",
       "8     rs79996746,COSV57674301\n",
       "9     rs79996746,COSV57674301\n",
       "10    rs79996746,COSV57674301\n",
       "11    rs79996746,COSV57674301\n",
       "12    rs79996746,COSV57674301\n",
       "13    rs79996746,COSV57674301\n",
       "14    rs79996746,COSV57674301\n",
       "15    rs79996746,COSV57674301\n",
       "16    rs79996746,COSV57674301\n",
       "17    rs79996746,COSV57674301\n",
       "18    rs79996746,COSV57674301\n",
       "19    rs12210856             \n",
       "20    rs12210856             \n",
       "21    rs12210856             \n",
       "22    rs12210856             \n",
       "23    rs12210856             \n",
       "24    rs12210856             \n",
       "25    rs12210856             \n",
       "26    rs12210856             \n",
       "27    rs12210856             \n",
       "28    rs12210856             \n",
       "29    rs12210856             \n",
       "30    rs12210856             \n",
       "⋮     ⋮                      \n",
       "37786 rs9825563              \n",
       "37787 rs9825563              \n",
       "37788 rs9825563              \n",
       "37789 rs9825563              \n",
       "37790 rs9825563              \n",
       "37791 rs75970639             \n",
       "37792 rs75970639             \n",
       "37793 rs75970639             \n",
       "37794 rs75970639             \n",
       "37795 rs75970639             \n",
       "37796 rs62267154             \n",
       "37797 rs62267154             \n",
       "37798 rs62267154             \n",
       "37799 rs62267154             \n",
       "37800 rs62267154             \n",
       "37801 rs150772796            \n",
       "37802 rs150772796            \n",
       "37803 rs150772796            \n",
       "37804 rs150772796            \n",
       "37805 rs150772796            \n",
       "37806 rs145438954            \n",
       "37807 rs145438954            \n",
       "37808 rs145438954            \n",
       "37809 rs145438954            \n",
       "37810 rs145438954            \n",
       "37811 rs112915369            \n",
       "37812 rs112915369            \n",
       "37813 rs112915369            \n",
       "37814 rs112915369            \n",
       "37815 rs112915369            \n",
       "      V14                                                                                                   \n",
       "1     IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   \n",
       "2     IMPACT=MODIFIER;DISTANCE=3458;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   \n",
       "3     IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   \n",
       "4     IMPACT=MODIFIER;DISTANCE=3320;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   \n",
       "5     IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   \n",
       "6     IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   \n",
       "7     IMPACT=MODIFIER;DISTANCE=3567;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   \n",
       "8     IMPACT=MODIFIER;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1                 \n",
       "9     IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   \n",
       "10    IMPACT=MODIFIER;DISTANCE=3674;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   \n",
       "11    IMPACT=MODIFIER;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1                 \n",
       "12    IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   \n",
       "13    IMPACT=MODIFIER;STRAND=1;FLAGS=cds_end_NF;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1\n",
       "14    IMPACT=MODIFIER;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1                 \n",
       "15    IMPACT=MODIFIER;DISTANCE=3693;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   \n",
       "16    IMPACT=MODIFIER;DISTANCE=3582;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   \n",
       "17    IMPACT=MODIFIER;DISTANCE=3697;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   \n",
       "18    IMPACT=MODIFIER;DISTANCE=3713;STRAND=1;MAX_AF=0.07796;MAX_AF_POPS=gnomADg_SAS;SOMATIC=0,1;PHENO=0,1   \n",
       "19    IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  \n",
       "20    IMPACT=MODIFIER;DISTANCE=2389;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  \n",
       "21    IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  \n",
       "22    IMPACT=MODIFIER;DISTANCE=2251;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  \n",
       "23    IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  \n",
       "24    IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  \n",
       "25    IMPACT=MODIFIER;DISTANCE=2498;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  \n",
       "26    IMPACT=MODIFIER;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                                \n",
       "27    IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  \n",
       "28    IMPACT=MODIFIER;DISTANCE=2605;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  \n",
       "29    IMPACT=MODIFIER;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                                \n",
       "30    IMPACT=MODIFIER;DISTANCE=2644;STRAND=1;MAX_AF=0.3517;MAX_AF_POPS=AFR                                  \n",
       "⋮     ⋮                                                                                                     \n",
       "37786 IMPACT=MODIFIER;DISTANCE=2321;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR                                 \n",
       "37787 IMPACT=MODIFIER;DISTANCE=2321;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR                                 \n",
       "37788 IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR                                               \n",
       "37789 IMPACT=MODIFIER;DISTANCE=2675;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR                                 \n",
       "37790 IMPACT=MODIFIER;DISTANCE=2321;STRAND=-1;MAX_AF=0.5204;MAX_AF_POPS=AFR                                 \n",
       "37791 IMPACT=MODIFIER;DISTANCE=2841;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS                                 \n",
       "37792 IMPACT=MODIFIER;DISTANCE=2841;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS                                 \n",
       "37793 IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS                                               \n",
       "37794 IMPACT=MODIFIER;DISTANCE=3195;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS                                 \n",
       "37795 IMPACT=MODIFIER;DISTANCE=2841;STRAND=-1;MAX_AF=0.1513;MAX_AF_POPS=SAS                                 \n",
       "37796 IMPACT=MODIFIER;DISTANCE=3204;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR                         \n",
       "37797 IMPACT=MODIFIER;DISTANCE=3204;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR                         \n",
       "37798 IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR                                       \n",
       "37799 IMPACT=MODIFIER;DISTANCE=3558;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR                         \n",
       "37800 IMPACT=MODIFIER;DISTANCE=3204;STRAND=-1;MAX_AF=0.2331;MAX_AF_POPS=gnomADg_AMR                         \n",
       "37801 IMPACT=MODIFIER;DISTANCE=3274;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR                                 \n",
       "37802 IMPACT=MODIFIER;DISTANCE=3274;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR                                 \n",
       "37803 IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR                                               \n",
       "37804 IMPACT=MODIFIER;DISTANCE=3628;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR                                 \n",
       "37805 IMPACT=MODIFIER;DISTANCE=3274;STRAND=-1;MAX_AF=0.2284;MAX_AF_POPS=AFR                                 \n",
       "37806 IMPACT=MODIFIER;DISTANCE=3430;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR                                 \n",
       "37807 IMPACT=MODIFIER;DISTANCE=3430;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR                                 \n",
       "37808 IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR                                               \n",
       "37809 IMPACT=MODIFIER;DISTANCE=3784;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR                                 \n",
       "37810 IMPACT=MODIFIER;DISTANCE=3430;STRAND=-1;MAX_AF=0.1415;MAX_AF_POPS=AFR                                 \n",
       "37811 IMPACT=MODIFIER;DISTANCE=3706;STRAND=-1;MAX_AF=0.08544;MAX_AF_POPS=gnomADg_MID                        \n",
       "37812 IMPACT=MODIFIER;DISTANCE=3706;STRAND=-1;MAX_AF=0.08544;MAX_AF_POPS=gnomADg_MID                        \n",
       "37813 IMPACT=MODIFIER;STRAND=-1;MAX_AF=0.08544;MAX_AF_POPS=gnomADg_MID                                      \n",
       "37814 IMPACT=MODIFIER;DISTANCE=4060;STRAND=-1;MAX_AF=0.08544;MAX_AF_POPS=gnomADg_MID                        \n",
       "37815 IMPACT=MODIFIER;DI                                                                                    "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "resultVEP"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "46738e39-baee-4156-8e08-ff16e6f2d0ea",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1mColumn specification\u001b[22m \u001b[36m──────────────────────────────────────────────────────────────\u001b[39m\n",
      "cols(\n",
      "  chr = \u001b[32mcol_double()\u001b[39m,\n",
      "  rs = \u001b[31mcol_character()\u001b[39m,\n",
      "  ps = \u001b[32mcol_double()\u001b[39m,\n",
      "  n_miss = \u001b[32mcol_double()\u001b[39m,\n",
      "  allele1 = \u001b[31mcol_character()\u001b[39m,\n",
      "  allele0 = \u001b[31mcol_character()\u001b[39m,\n",
      "  af = \u001b[32mcol_double()\u001b[39m,\n",
      "  logl_H1 = \u001b[32mcol_double()\u001b[39m,\n",
      "  l_mle = \u001b[32mcol_double()\u001b[39m,\n",
      "  p_lrt = \u001b[32mcol_double()\u001b[39m\n",
      ")\n",
      "\n"
     ]
    }
   ],
   "source": [
    "resultGEMMA <- read_table(\"output/GWASresults.lmm.assoc.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "42075bd5-ff1d-4023-b34d-6db290b44bc3",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A spec_tbl_df: 482 × 10</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>chr</th><th scope=col>rs</th><th scope=col>ps</th><th scope=col>n_miss</th><th scope=col>allele1</th><th scope=col>allele0</th><th scope=col>af</th><th scope=col>logl_H1</th><th scope=col>l_mle</th><th scope=col>p_lrt</th></tr>\n",
       "\t<tr><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><td>1</td><td>.</td><td> 28810675</td><td>0</td><td>G                                                                                      </td><td>A </td><td>0.105</td><td>-10.468100</td><td>1e+05</td><td>0.45864360</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28810768</td><td>0</td><td>A                                                                                      </td><td>G </td><td>0.105</td><td>-10.468100</td><td>1e+05</td><td>0.45864360</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28812463</td><td>0</td><td>T                                                                                      </td><td>G </td><td>0.763</td><td>-10.645140</td><td>1e+05</td><td>0.65867480</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28812628</td><td>0</td><td>G                                                                                      </td><td>C </td><td>0.184</td><td>-10.119090</td><td>1e+05</td><td>0.26408160</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28813896</td><td>0</td><td>T                                                                                      </td><td>G </td><td>0.105</td><td>-10.468100</td><td>1e+05</td><td>0.45864360</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28813907</td><td>0</td><td>C                                                                                      </td><td>T </td><td>0.105</td><td>-10.468100</td><td>1e+05</td><td>0.45864360</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28817718</td><td>0</td><td>G                                                                                      </td><td>T </td><td>0.105</td><td>-10.468100</td><td>1e+05</td><td>0.45864360</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28817739</td><td>0</td><td>C                                                                                      </td><td>G </td><td>0.105</td><td>-10.468100</td><td>1e+05</td><td>0.45864360</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28822623</td><td>0</td><td>A                                                                                      </td><td>T </td><td>0.263</td><td>-10.732210</td><td>1e+05</td><td>0.88480080</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28832616</td><td>0</td><td>C                                                                                      </td><td>CA</td><td>0.500</td><td>-10.688950</td><td>1e+05</td><td>0.74300190</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28838049</td><td>0</td><td>G                                                                                      </td><td>T </td><td>0.500</td><td>-10.688950</td><td>1e+05</td><td>0.74300190</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28838070</td><td>0</td><td>A                                                                                      </td><td>G </td><td>0.263</td><td>-10.732210</td><td>1e+05</td><td>0.88480080</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28839811</td><td>0</td><td>A                                                                                      </td><td>C </td><td>0.763</td><td>-10.645140</td><td>1e+05</td><td>0.65867480</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28843366</td><td>0</td><td>T                                                                                      </td><td>C </td><td>0.053</td><td> -9.441112</td><td>1e+05</td><td>0.10664900</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28844732</td><td>0</td><td>T                                                                                      </td><td>C </td><td>0.053</td><td> -9.441112</td><td>1e+05</td><td>0.10664900</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28844750</td><td>0</td><td>C                                                                                      </td><td>T </td><td>0.053</td><td> -9.441112</td><td>1e+05</td><td>0.10664900</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28847976</td><td>0</td><td>CG                                                                                     </td><td>C </td><td>0.053</td><td>-10.603440</td><td>1e+05</td><td>0.59765860</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28848861</td><td>0</td><td>G                                                                                      </td><td>A </td><td>0.737</td><td>-10.576860</td><td>1e+05</td><td>0.56466830</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28868306</td><td>0</td><td>G                                                                                      </td><td>A </td><td>0.579</td><td> -8.153650</td><td>1e+05</td><td>0.02287315</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td> 28868389</td><td>0</td><td>T                                                                                      </td><td>C </td><td>0.184</td><td> -9.285249</td><td>1e+05</td><td>0.08776407</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td>224613884</td><td>0</td><td>GT                                                                                     </td><td>G </td><td>0.053</td><td> -8.393842</td><td>1e+05</td><td>0.03020251</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td>224618472</td><td>0</td><td>C                                                                                      </td><td>G </td><td>0.684</td><td> -9.205706</td><td>1e+05</td><td>0.07955344</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td>224627075</td><td>0</td><td>G                                                                                      </td><td>A </td><td>0.079</td><td> -8.037921</td><td>1e+05</td><td>0.02002664</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td>224633220</td><td>0</td><td>A                                                                                      </td><td>G </td><td>0.289</td><td>-10.590850</td><td>1e+05</td><td>0.58156830</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td>224633972</td><td>0</td><td>GGCTGAGGGTCAGCGACACATCGTCGAGAATGTAAGCGCCTGTCCGCCACCGGGCCCCCTCCGGGAGAACCGGCCATGCGAAGTCGT</td><td>G </td><td>0.053</td><td>-10.696080</td><td>1e+05</td><td>0.76007180</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td>224648541</td><td>0</td><td>A                                                                                      </td><td>G </td><td>0.053</td><td>-10.300630</td><td>1e+05</td><td>0.34706420</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td>224648594</td><td>0</td><td>T                                                                                      </td><td>C </td><td>0.053</td><td>-10.300630</td><td>1e+05</td><td>0.34706420</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td>224649649</td><td>0</td><td>C                                                                                      </td><td>T </td><td>0.105</td><td>-10.731110</td><td>1e+05</td><td>0.87897270</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td>224657071</td><td>0</td><td>T                                                                                      </td><td>C </td><td>0.053</td><td>-10.300630</td><td>1e+05</td><td>0.34706420</td></tr>\n",
       "\t<tr><td>1</td><td>.</td><td>224659048</td><td>0</td><td>A                                                                                      </td><td>G </td><td>0.289</td><td>-10.590850</td><td>1e+05</td><td>0.58156830</td></tr>\n",
       "\t<tr><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td></tr>\n",
       "\t<tr><td>15</td><td>.</td><td>93083603</td><td>0</td><td>G                                                                                                        </td><td>A    </td><td>0.079</td><td>-10.169340</td><td>1e+05</td><td>0.2842344000</td></tr>\n",
       "\t<tr><td>15</td><td>.</td><td>93083620</td><td>0</td><td>T                                                                                                        </td><td>C    </td><td>0.105</td><td>-10.658940</td><td>1e+05</td><td>0.6823087000</td></tr>\n",
       "\t<tr><td>15</td><td>.</td><td>93085681</td><td>0</td><td>G                                                                                                        </td><td>A    </td><td>0.132</td><td> -9.586946</td><td>1e+05</td><td>0.1284184000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79796588</td><td>0</td><td>A                                                                                                        </td><td>G    </td><td>0.289</td><td>-10.606420</td><td>1e+05</td><td>0.6016047000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79803134</td><td>0</td><td>T                                                                                                        </td><td>C    </td><td>0.158</td><td>-10.263390</td><td>1e+05</td><td>0.3275301000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79806123</td><td>0</td><td>T                                                                                                        </td><td>C    </td><td>0.079</td><td> -8.244272</td><td>1e+05</td><td>0.0253932200</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79818385</td><td>0</td><td>A                                                                                                        </td><td>G    </td><td>0.053</td><td>-10.714880</td><td>1e+05</td><td>0.8134935000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79818986</td><td>0</td><td>A                                                                                                        </td><td>G    </td><td>0.105</td><td>-10.646590</td><td>1e+05</td><td>0.6610606000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79819038</td><td>0</td><td>G                                                                                                        </td><td>C    </td><td>0.263</td><td>-10.114500</td><td>1e+05</td><td>0.2623302000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79832363</td><td>0</td><td>G                                                                                                        </td><td>A    </td><td>0.105</td><td> -8.552956</td><td>1e+05</td><td>0.0363736500</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79834238</td><td>0</td><td>G                                                                                                        </td><td>A    </td><td>0.947</td><td>-10.696080</td><td>1e+05</td><td>0.7600718000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79834239</td><td>0</td><td>A                                                                                                        </td><td>G    </td><td>0.105</td><td> -8.552956</td><td>1e+05</td><td>0.0363736500</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79846478</td><td>0</td><td>C                                                                                                        </td><td>CAT  </td><td>0.316</td><td> -5.599546</td><td>1e+05</td><td>0.0013402000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79849355</td><td>0</td><td>T                                                                                                        </td><td>C    </td><td>0.447</td><td>-10.624620</td><td>1e+05</td><td>0.6269828000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79852430</td><td>0</td><td>AATTCCCTGCCCGATAACGCGGACCACGGCACGGCCGTCGCCCAGATCCTCGTGGCGCCCAAGTTCGGTGTCGCCCCCGGCGCGACCCTGTACACCTACGACCTG</td><td>A    </td><td>0.053</td><td>-10.696080</td><td>1e+05</td><td>0.7600718000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79862356</td><td>0</td><td>C                                                                                                        </td><td>A    </td><td>0.368</td><td>-10.199840</td><td>1e+05</td><td>0.2974179000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79869217</td><td>0</td><td>T                                                                                                        </td><td>C    </td><td>0.105</td><td> -9.005470</td><td>1e+05</td><td>0.0623225500</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79869482</td><td>0</td><td>A                                                                                                        </td><td>G    </td><td>0.105</td><td> -9.005470</td><td>1e+05</td><td>0.0623225500</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79876250</td><td>0</td><td>G                                                                                                        </td><td>A    </td><td>0.921</td><td>-10.096590</td><td>1e+05</td><td>0.2556367000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79876519</td><td>0</td><td>G                                                                                                        </td><td>A    </td><td>0.158</td><td>-10.626240</td><td>1e+05</td><td>0.6293590000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79876659</td><td>0</td><td>G                                                                                                        </td><td>A    </td><td>0.895</td><td>-10.556750</td><td>1e+05</td><td>0.5419614000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79877424</td><td>0</td><td>C                                                                                                        </td><td>T    </td><td>0.737</td><td> -9.747092</td><td>1e+05</td><td>0.1582126000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79877493</td><td>0</td><td>A                                                                                                        </td><td>G    </td><td>0.737</td><td> -9.747092</td><td>1e+05</td><td>0.1582126000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79887406</td><td>0</td><td>A                                                                                                        </td><td>G    </td><td>0.342</td><td> -5.279159</td><td>1e+05</td><td>0.0009476796</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79893897</td><td>0</td><td>A                                                                                                        </td><td>G    </td><td>0.184</td><td>-10.735120</td><td>1e+05</td><td>0.9019485000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79893908</td><td>0</td><td>T                                                                                                        </td><td>TA   </td><td>0.184</td><td>-10.735120</td><td>1e+05</td><td>0.9019485000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79893916</td><td>0</td><td>G                                                                                                        </td><td>A    </td><td>0.184</td><td>-10.735120</td><td>1e+05</td><td>0.9019485000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79893922</td><td>0</td><td>A                                                                                                        </td><td>G    </td><td>0.184</td><td>-10.735120</td><td>1e+05</td><td>0.9019485000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79895807</td><td>0</td><td>A                                                                                                        </td><td>C    </td><td>0.158</td><td>-10.626240</td><td>1e+05</td><td>0.6293590000</td></tr>\n",
       "\t<tr><td>18</td><td>.</td><td>79895817</td><td>0</td><td>T                                                                                                        </td><td>TGATA</td><td>0.053</td><td> -9.303518</td><td>1e+05</td><td>0.0897765000</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A spec\\_tbl\\_df: 482 × 10\n",
       "\\begin{tabular}{llllllllll}\n",
       " chr & rs & ps & n\\_miss & allele1 & allele0 & af & logl\\_H1 & l\\_mle & p\\_lrt\\\\\n",
       " <dbl> & <chr> & <dbl> & <dbl> & <chr> & <chr> & <dbl> & <dbl> & <dbl> & <dbl>\\\\\n",
       "\\hline\n",
       "\t 1 & . &  28810675 & 0 & G                                                                                       & A  & 0.105 & -10.468100 & 1e+05 & 0.45864360\\\\\n",
       "\t 1 & . &  28810768 & 0 & A                                                                                       & G  & 0.105 & -10.468100 & 1e+05 & 0.45864360\\\\\n",
       "\t 1 & . &  28812463 & 0 & T                                                                                       & G  & 0.763 & -10.645140 & 1e+05 & 0.65867480\\\\\n",
       "\t 1 & . &  28812628 & 0 & G                                                                                       & C  & 0.184 & -10.119090 & 1e+05 & 0.26408160\\\\\n",
       "\t 1 & . &  28813896 & 0 & T                                                                                       & G  & 0.105 & -10.468100 & 1e+05 & 0.45864360\\\\\n",
       "\t 1 & . &  28813907 & 0 & C                                                                                       & T  & 0.105 & -10.468100 & 1e+05 & 0.45864360\\\\\n",
       "\t 1 & . &  28817718 & 0 & G                                                                                       & T  & 0.105 & -10.468100 & 1e+05 & 0.45864360\\\\\n",
       "\t 1 & . &  28817739 & 0 & C                                                                                       & G  & 0.105 & -10.468100 & 1e+05 & 0.45864360\\\\\n",
       "\t 1 & . &  28822623 & 0 & A                                                                                       & T  & 0.263 & -10.732210 & 1e+05 & 0.88480080\\\\\n",
       "\t 1 & . &  28832616 & 0 & C                                                                                       & CA & 0.500 & -10.688950 & 1e+05 & 0.74300190\\\\\n",
       "\t 1 & . &  28838049 & 0 & G                                                                                       & T  & 0.500 & -10.688950 & 1e+05 & 0.74300190\\\\\n",
       "\t 1 & . &  28838070 & 0 & A                                                                                       & G  & 0.263 & -10.732210 & 1e+05 & 0.88480080\\\\\n",
       "\t 1 & . &  28839811 & 0 & A                                                                                       & C  & 0.763 & -10.645140 & 1e+05 & 0.65867480\\\\\n",
       "\t 1 & . &  28843366 & 0 & T                                                                                       & C  & 0.053 &  -9.441112 & 1e+05 & 0.10664900\\\\\n",
       "\t 1 & . &  28844732 & 0 & T                                                                                       & C  & 0.053 &  -9.441112 & 1e+05 & 0.10664900\\\\\n",
       "\t 1 & . &  28844750 & 0 & C                                                                                       & T  & 0.053 &  -9.441112 & 1e+05 & 0.10664900\\\\\n",
       "\t 1 & . &  28847976 & 0 & CG                                                                                      & C  & 0.053 & -10.603440 & 1e+05 & 0.59765860\\\\\n",
       "\t 1 & . &  28848861 & 0 & G                                                                                       & A  & 0.737 & -10.576860 & 1e+05 & 0.56466830\\\\\n",
       "\t 1 & . &  28868306 & 0 & G                                                                                       & A  & 0.579 &  -8.153650 & 1e+05 & 0.02287315\\\\\n",
       "\t 1 & . &  28868389 & 0 & T                                                                                       & C  & 0.184 &  -9.285249 & 1e+05 & 0.08776407\\\\\n",
       "\t 1 & . & 224613884 & 0 & GT                                                                                      & G  & 0.053 &  -8.393842 & 1e+05 & 0.03020251\\\\\n",
       "\t 1 & . & 224618472 & 0 & C                                                                                       & G  & 0.684 &  -9.205706 & 1e+05 & 0.07955344\\\\\n",
       "\t 1 & . & 224627075 & 0 & G                                                                                       & A  & 0.079 &  -8.037921 & 1e+05 & 0.02002664\\\\\n",
       "\t 1 & . & 224633220 & 0 & A                                                                                       & G  & 0.289 & -10.590850 & 1e+05 & 0.58156830\\\\\n",
       "\t 1 & . & 224633972 & 0 & GGCTGAGGGTCAGCGACACATCGTCGAGAATGTAAGCGCCTGTCCGCCACCGGGCCCCCTCCGGGAGAACCGGCCATGCGAAGTCGT & G  & 0.053 & -10.696080 & 1e+05 & 0.76007180\\\\\n",
       "\t 1 & . & 224648541 & 0 & A                                                                                       & G  & 0.053 & -10.300630 & 1e+05 & 0.34706420\\\\\n",
       "\t 1 & . & 224648594 & 0 & T                                                                                       & C  & 0.053 & -10.300630 & 1e+05 & 0.34706420\\\\\n",
       "\t 1 & . & 224649649 & 0 & C                                                                                       & T  & 0.105 & -10.731110 & 1e+05 & 0.87897270\\\\\n",
       "\t 1 & . & 224657071 & 0 & T                                                                                       & C  & 0.053 & -10.300630 & 1e+05 & 0.34706420\\\\\n",
       "\t 1 & . & 224659048 & 0 & A                                                                                       & G  & 0.289 & -10.590850 & 1e+05 & 0.58156830\\\\\n",
       "\t ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮ & ⋮\\\\\n",
       "\t 15 & . & 93083603 & 0 & G                                                                                                         & A     & 0.079 & -10.169340 & 1e+05 & 0.2842344000\\\\\n",
       "\t 15 & . & 93083620 & 0 & T                                                                                                         & C     & 0.105 & -10.658940 & 1e+05 & 0.6823087000\\\\\n",
       "\t 15 & . & 93085681 & 0 & G                                                                                                         & A     & 0.132 &  -9.586946 & 1e+05 & 0.1284184000\\\\\n",
       "\t 18 & . & 79796588 & 0 & A                                                                                                         & G     & 0.289 & -10.606420 & 1e+05 & 0.6016047000\\\\\n",
       "\t 18 & . & 79803134 & 0 & T                                                                                                         & C     & 0.158 & -10.263390 & 1e+05 & 0.3275301000\\\\\n",
       "\t 18 & . & 79806123 & 0 & T                                                                                                         & C     & 0.079 &  -8.244272 & 1e+05 & 0.0253932200\\\\\n",
       "\t 18 & . & 79818385 & 0 & A                                                                                                         & G     & 0.053 & -10.714880 & 1e+05 & 0.8134935000\\\\\n",
       "\t 18 & . & 79818986 & 0 & A                                                                                                         & G     & 0.105 & -10.646590 & 1e+05 & 0.6610606000\\\\\n",
       "\t 18 & . & 79819038 & 0 & G                                                                                                         & C     & 0.263 & -10.114500 & 1e+05 & 0.2623302000\\\\\n",
       "\t 18 & . & 79832363 & 0 & G                                                                                                         & A     & 0.105 &  -8.552956 & 1e+05 & 0.0363736500\\\\\n",
       "\t 18 & . & 79834238 & 0 & G                                                                                                         & A     & 0.947 & -10.696080 & 1e+05 & 0.7600718000\\\\\n",
       "\t 18 & . & 79834239 & 0 & A                                                                                                         & G     & 0.105 &  -8.552956 & 1e+05 & 0.0363736500\\\\\n",
       "\t 18 & . & 79846478 & 0 & C                                                                                                         & CAT   & 0.316 &  -5.599546 & 1e+05 & 0.0013402000\\\\\n",
       "\t 18 & . & 79849355 & 0 & T                                                                                                         & C     & 0.447 & -10.624620 & 1e+05 & 0.6269828000\\\\\n",
       "\t 18 & . & 79852430 & 0 & AATTCCCTGCCCGATAACGCGGACCACGGCACGGCCGTCGCCCAGATCCTCGTGGCGCCCAAGTTCGGTGTCGCCCCCGGCGCGACCCTGTACACCTACGACCTG & A     & 0.053 & -10.696080 & 1e+05 & 0.7600718000\\\\\n",
       "\t 18 & . & 79862356 & 0 & C                                                                                                         & A     & 0.368 & -10.199840 & 1e+05 & 0.2974179000\\\\\n",
       "\t 18 & . & 79869217 & 0 & T                                                                                                         & C     & 0.105 &  -9.005470 & 1e+05 & 0.0623225500\\\\\n",
       "\t 18 & . & 79869482 & 0 & A                                                                                                         & G     & 0.105 &  -9.005470 & 1e+05 & 0.0623225500\\\\\n",
       "\t 18 & . & 79876250 & 0 & G                                                                                                         & A     & 0.921 & -10.096590 & 1e+05 & 0.2556367000\\\\\n",
       "\t 18 & . & 79876519 & 0 & G                                                                                                         & A     & 0.158 & -10.626240 & 1e+05 & 0.6293590000\\\\\n",
       "\t 18 & . & 79876659 & 0 & G                                                                                                         & A     & 0.895 & -10.556750 & 1e+05 & 0.5419614000\\\\\n",
       "\t 18 & . & 79877424 & 0 & C                                                                                                         & T     & 0.737 &  -9.747092 & 1e+05 & 0.1582126000\\\\\n",
       "\t 18 & . & 79877493 & 0 & A                                                                                                         & G     & 0.737 &  -9.747092 & 1e+05 & 0.1582126000\\\\\n",
       "\t 18 & . & 79887406 & 0 & A                                                                                                         & G     & 0.342 &  -5.279159 & 1e+05 & 0.0009476796\\\\\n",
       "\t 18 & . & 79893897 & 0 & A                                                                                                         & G     & 0.184 & -10.735120 & 1e+05 & 0.9019485000\\\\\n",
       "\t 18 & . & 79893908 & 0 & T                                                                                                         & TA    & 0.184 & -10.735120 & 1e+05 & 0.9019485000\\\\\n",
       "\t 18 & . & 79893916 & 0 & G                                                                                                         & A     & 0.184 & -10.735120 & 1e+05 & 0.9019485000\\\\\n",
       "\t 18 & . & 79893922 & 0 & A                                                                                                         & G     & 0.184 & -10.735120 & 1e+05 & 0.9019485000\\\\\n",
       "\t 18 & . & 79895807 & 0 & A                                                                                                         & C     & 0.158 & -10.626240 & 1e+05 & 0.6293590000\\\\\n",
       "\t 18 & . & 79895817 & 0 & T                                                                                                         & TGATA & 0.053 &  -9.303518 & 1e+05 & 0.0897765000\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A spec_tbl_df: 482 × 10\n",
       "\n",
       "| chr &lt;dbl&gt; | rs &lt;chr&gt; | ps &lt;dbl&gt; | n_miss &lt;dbl&gt; | allele1 &lt;chr&gt; | allele0 &lt;chr&gt; | af &lt;dbl&gt; | logl_H1 &lt;dbl&gt; | l_mle &lt;dbl&gt; | p_lrt &lt;dbl&gt; |\n",
       "|---|---|---|---|---|---|---|---|---|---|\n",
       "| 1 | . |  28810675 | 0 | G                                                                                       | A  | 0.105 | -10.468100 | 1e+05 | 0.45864360 |\n",
       "| 1 | . |  28810768 | 0 | A                                                                                       | G  | 0.105 | -10.468100 | 1e+05 | 0.45864360 |\n",
       "| 1 | . |  28812463 | 0 | T                                                                                       | G  | 0.763 | -10.645140 | 1e+05 | 0.65867480 |\n",
       "| 1 | . |  28812628 | 0 | G                                                                                       | C  | 0.184 | -10.119090 | 1e+05 | 0.26408160 |\n",
       "| 1 | . |  28813896 | 0 | T                                                                                       | G  | 0.105 | -10.468100 | 1e+05 | 0.45864360 |\n",
       "| 1 | . |  28813907 | 0 | C                                                                                       | T  | 0.105 | -10.468100 | 1e+05 | 0.45864360 |\n",
       "| 1 | . |  28817718 | 0 | G                                                                                       | T  | 0.105 | -10.468100 | 1e+05 | 0.45864360 |\n",
       "| 1 | . |  28817739 | 0 | C                                                                                       | G  | 0.105 | -10.468100 | 1e+05 | 0.45864360 |\n",
       "| 1 | . |  28822623 | 0 | A                                                                                       | T  | 0.263 | -10.732210 | 1e+05 | 0.88480080 |\n",
       "| 1 | . |  28832616 | 0 | C                                                                                       | CA | 0.500 | -10.688950 | 1e+05 | 0.74300190 |\n",
       "| 1 | . |  28838049 | 0 | G                                                                                       | T  | 0.500 | -10.688950 | 1e+05 | 0.74300190 |\n",
       "| 1 | . |  28838070 | 0 | A                                                                                       | G  | 0.263 | -10.732210 | 1e+05 | 0.88480080 |\n",
       "| 1 | . |  28839811 | 0 | A                                                                                       | C  | 0.763 | -10.645140 | 1e+05 | 0.65867480 |\n",
       "| 1 | . |  28843366 | 0 | T                                                                                       | C  | 0.053 |  -9.441112 | 1e+05 | 0.10664900 |\n",
       "| 1 | . |  28844732 | 0 | T                                                                                       | C  | 0.053 |  -9.441112 | 1e+05 | 0.10664900 |\n",
       "| 1 | . |  28844750 | 0 | C                                                                                       | T  | 0.053 |  -9.441112 | 1e+05 | 0.10664900 |\n",
       "| 1 | . |  28847976 | 0 | CG                                                                                      | C  | 0.053 | -10.603440 | 1e+05 | 0.59765860 |\n",
       "| 1 | . |  28848861 | 0 | G                                                                                       | A  | 0.737 | -10.576860 | 1e+05 | 0.56466830 |\n",
       "| 1 | . |  28868306 | 0 | G                                                                                       | A  | 0.579 |  -8.153650 | 1e+05 | 0.02287315 |\n",
       "| 1 | . |  28868389 | 0 | T                                                                                       | C  | 0.184 |  -9.285249 | 1e+05 | 0.08776407 |\n",
       "| 1 | . | 224613884 | 0 | GT                                                                                      | G  | 0.053 |  -8.393842 | 1e+05 | 0.03020251 |\n",
       "| 1 | . | 224618472 | 0 | C                                                                                       | G  | 0.684 |  -9.205706 | 1e+05 | 0.07955344 |\n",
       "| 1 | . | 224627075 | 0 | G                                                                                       | A  | 0.079 |  -8.037921 | 1e+05 | 0.02002664 |\n",
       "| 1 | . | 224633220 | 0 | A                                                                                       | G  | 0.289 | -10.590850 | 1e+05 | 0.58156830 |\n",
       "| 1 | . | 224633972 | 0 | GGCTGAGGGTCAGCGACACATCGTCGAGAATGTAAGCGCCTGTCCGCCACCGGGCCCCCTCCGGGAGAACCGGCCATGCGAAGTCGT | G  | 0.053 | -10.696080 | 1e+05 | 0.76007180 |\n",
       "| 1 | . | 224648541 | 0 | A                                                                                       | G  | 0.053 | -10.300630 | 1e+05 | 0.34706420 |\n",
       "| 1 | . | 224648594 | 0 | T                                                                                       | C  | 0.053 | -10.300630 | 1e+05 | 0.34706420 |\n",
       "| 1 | . | 224649649 | 0 | C                                                                                       | T  | 0.105 | -10.731110 | 1e+05 | 0.87897270 |\n",
       "| 1 | . | 224657071 | 0 | T                                                                                       | C  | 0.053 | -10.300630 | 1e+05 | 0.34706420 |\n",
       "| 1 | . | 224659048 | 0 | A                                                                                       | G  | 0.289 | -10.590850 | 1e+05 | 0.58156830 |\n",
       "| ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ | ⋮ |\n",
       "| 15 | . | 93083603 | 0 | G                                                                                                         | A     | 0.079 | -10.169340 | 1e+05 | 0.2842344000 |\n",
       "| 15 | . | 93083620 | 0 | T                                                                                                         | C     | 0.105 | -10.658940 | 1e+05 | 0.6823087000 |\n",
       "| 15 | . | 93085681 | 0 | G                                                                                                         | A     | 0.132 |  -9.586946 | 1e+05 | 0.1284184000 |\n",
       "| 18 | . | 79796588 | 0 | A                                                                                                         | G     | 0.289 | -10.606420 | 1e+05 | 0.6016047000 |\n",
       "| 18 | . | 79803134 | 0 | T                                                                                                         | C     | 0.158 | -10.263390 | 1e+05 | 0.3275301000 |\n",
       "| 18 | . | 79806123 | 0 | T                                                                                                         | C     | 0.079 |  -8.244272 | 1e+05 | 0.0253932200 |\n",
       "| 18 | . | 79818385 | 0 | A                                                                                                         | G     | 0.053 | -10.714880 | 1e+05 | 0.8134935000 |\n",
       "| 18 | . | 79818986 | 0 | A                                                                                                         | G     | 0.105 | -10.646590 | 1e+05 | 0.6610606000 |\n",
       "| 18 | . | 79819038 | 0 | G                                                                                                         | C     | 0.263 | -10.114500 | 1e+05 | 0.2623302000 |\n",
       "| 18 | . | 79832363 | 0 | G                                                                                                         | A     | 0.105 |  -8.552956 | 1e+05 | 0.0363736500 |\n",
       "| 18 | . | 79834238 | 0 | G                                                                                                         | A     | 0.947 | -10.696080 | 1e+05 | 0.7600718000 |\n",
       "| 18 | . | 79834239 | 0 | A                                                                                                         | G     | 0.105 |  -8.552956 | 1e+05 | 0.0363736500 |\n",
       "| 18 | . | 79846478 | 0 | C                                                                                                         | CAT   | 0.316 |  -5.599546 | 1e+05 | 0.0013402000 |\n",
       "| 18 | . | 79849355 | 0 | T                                                                                                         | C     | 0.447 | -10.624620 | 1e+05 | 0.6269828000 |\n",
       "| 18 | . | 79852430 | 0 | AATTCCCTGCCCGATAACGCGGACCACGGCACGGCCGTCGCCCAGATCCTCGTGGCGCCCAAGTTCGGTGTCGCCCCCGGCGCGACCCTGTACACCTACGACCTG | A     | 0.053 | -10.696080 | 1e+05 | 0.7600718000 |\n",
       "| 18 | . | 79862356 | 0 | C                                                                                                         | A     | 0.368 | -10.199840 | 1e+05 | 0.2974179000 |\n",
       "| 18 | . | 79869217 | 0 | T                                                                                                         | C     | 0.105 |  -9.005470 | 1e+05 | 0.0623225500 |\n",
       "| 18 | . | 79869482 | 0 | A                                                                                                         | G     | 0.105 |  -9.005470 | 1e+05 | 0.0623225500 |\n",
       "| 18 | . | 79876250 | 0 | G                                                                                                         | A     | 0.921 | -10.096590 | 1e+05 | 0.2556367000 |\n",
       "| 18 | . | 79876519 | 0 | G                                                                                                         | A     | 0.158 | -10.626240 | 1e+05 | 0.6293590000 |\n",
       "| 18 | . | 79876659 | 0 | G                                                                                                         | A     | 0.895 | -10.556750 | 1e+05 | 0.5419614000 |\n",
       "| 18 | . | 79877424 | 0 | C                                                                                                         | T     | 0.737 |  -9.747092 | 1e+05 | 0.1582126000 |\n",
       "| 18 | . | 79877493 | 0 | A                                                                                                         | G     | 0.737 |  -9.747092 | 1e+05 | 0.1582126000 |\n",
       "| 18 | . | 79887406 | 0 | A                                                                                                         | G     | 0.342 |  -5.279159 | 1e+05 | 0.0009476796 |\n",
       "| 18 | . | 79893897 | 0 | A                                                                                                         | G     | 0.184 | -10.735120 | 1e+05 | 0.9019485000 |\n",
       "| 18 | . | 79893908 | 0 | T                                                                                                         | TA    | 0.184 | -10.735120 | 1e+05 | 0.9019485000 |\n",
       "| 18 | . | 79893916 | 0 | G                                                                                                         | A     | 0.184 | -10.735120 | 1e+05 | 0.9019485000 |\n",
       "| 18 | . | 79893922 | 0 | A                                                                                                         | G     | 0.184 | -10.735120 | 1e+05 | 0.9019485000 |\n",
       "| 18 | . | 79895807 | 0 | A                                                                                                         | C     | 0.158 | -10.626240 | 1e+05 | 0.6293590000 |\n",
       "| 18 | . | 79895817 | 0 | T                                                                                                         | TGATA | 0.053 |  -9.303518 | 1e+05 | 0.0897765000 |\n",
       "\n"
      ],
      "text/plain": [
       "    chr rs ps        n_miss\n",
       "1   1   .   28810675 0     \n",
       "2   1   .   28810768 0     \n",
       "3   1   .   28812463 0     \n",
       "4   1   .   28812628 0     \n",
       "5   1   .   28813896 0     \n",
       "6   1   .   28813907 0     \n",
       "7   1   .   28817718 0     \n",
       "8   1   .   28817739 0     \n",
       "9   1   .   28822623 0     \n",
       "10  1   .   28832616 0     \n",
       "11  1   .   28838049 0     \n",
       "12  1   .   28838070 0     \n",
       "13  1   .   28839811 0     \n",
       "14  1   .   28843366 0     \n",
       "15  1   .   28844732 0     \n",
       "16  1   .   28844750 0     \n",
       "17  1   .   28847976 0     \n",
       "18  1   .   28848861 0     \n",
       "19  1   .   28868306 0     \n",
       "20  1   .   28868389 0     \n",
       "21  1   .  224613884 0     \n",
       "22  1   .  224618472 0     \n",
       "23  1   .  224627075 0     \n",
       "24  1   .  224633220 0     \n",
       "25  1   .  224633972 0     \n",
       "26  1   .  224648541 0     \n",
       "27  1   .  224648594 0     \n",
       "28  1   .  224649649 0     \n",
       "29  1   .  224657071 0     \n",
       "30  1   .  224659048 0     \n",
       "⋮   ⋮   ⋮  ⋮         ⋮     \n",
       "453 15  .  93083603  0     \n",
       "454 15  .  93083620  0     \n",
       "455 15  .  93085681  0     \n",
       "456 18  .  79796588  0     \n",
       "457 18  .  79803134  0     \n",
       "458 18  .  79806123  0     \n",
       "459 18  .  79818385  0     \n",
       "460 18  .  79818986  0     \n",
       "461 18  .  79819038  0     \n",
       "462 18  .  79832363  0     \n",
       "463 18  .  79834238  0     \n",
       "464 18  .  79834239  0     \n",
       "465 18  .  79846478  0     \n",
       "466 18  .  79849355  0     \n",
       "467 18  .  79852430  0     \n",
       "468 18  .  79862356  0     \n",
       "469 18  .  79869217  0     \n",
       "470 18  .  79869482  0     \n",
       "471 18  .  79876250  0     \n",
       "472 18  .  79876519  0     \n",
       "473 18  .  79876659  0     \n",
       "474 18  .  79877424  0     \n",
       "475 18  .  79877493  0     \n",
       "476 18  .  79887406  0     \n",
       "477 18  .  79893897  0     \n",
       "478 18  .  79893908  0     \n",
       "479 18  .  79893916  0     \n",
       "480 18  .  79893922  0     \n",
       "481 18  .  79895807  0     \n",
       "482 18  .  79895817  0     \n",
       "    allele1                                                                                                  \n",
       "1   G                                                                                                        \n",
       "2   A                                                                                                        \n",
       "3   T                                                                                                        \n",
       "4   G                                                                                                        \n",
       "5   T                                                                                                        \n",
       "6   C                                                                                                        \n",
       "7   G                                                                                                        \n",
       "8   C                                                                                                        \n",
       "9   A                                                                                                        \n",
       "10  C                                                                                                        \n",
       "11  G                                                                                                        \n",
       "12  A                                                                                                        \n",
       "13  A                                                                                                        \n",
       "14  T                                                                                                        \n",
       "15  T                                                                                                        \n",
       "16  C                                                                                                        \n",
       "17  CG                                                                                                       \n",
       "18  G                                                                                                        \n",
       "19  G                                                                                                        \n",
       "20  T                                                                                                        \n",
       "21  GT                                                                                                       \n",
       "22  C                                                                                                        \n",
       "23  G                                                                                                        \n",
       "24  A                                                                                                        \n",
       "25  GGCTGAGGGTCAGCGACACATCGTCGAGAATGTAAGCGCCTGTCCGCCACCGGGCCCCCTCCGGGAGAACCGGCCATGCGAAGTCGT                  \n",
       "26  A                                                                                                        \n",
       "27  T                                                                                                        \n",
       "28  C                                                                                                        \n",
       "29  T                                                                                                        \n",
       "30  A                                                                                                        \n",
       "⋮   ⋮                                                                                                        \n",
       "453 G                                                                                                        \n",
       "454 T                                                                                                        \n",
       "455 G                                                                                                        \n",
       "456 A                                                                                                        \n",
       "457 T                                                                                                        \n",
       "458 T                                                                                                        \n",
       "459 A                                                                                                        \n",
       "460 A                                                                                                        \n",
       "461 G                                                                                                        \n",
       "462 G                                                                                                        \n",
       "463 G                                                                                                        \n",
       "464 A                                                                                                        \n",
       "465 C                                                                                                        \n",
       "466 T                                                                                                        \n",
       "467 AATTCCCTGCCCGATAACGCGGACCACGGCACGGCCGTCGCCCAGATCCTCGTGGCGCCCAAGTTCGGTGTCGCCCCCGGCGCGACCCTGTACACCTACGACCTG\n",
       "468 C                                                                                                        \n",
       "469 T                                                                                                        \n",
       "470 A                                                                                                        \n",
       "471 G                                                                                                        \n",
       "472 G                                                                                                        \n",
       "473 G                                                                                                        \n",
       "474 C                                                                                                        \n",
       "475 A                                                                                                        \n",
       "476 A                                                                                                        \n",
       "477 A                                                                                                        \n",
       "478 T                                                                                                        \n",
       "479 G                                                                                                        \n",
       "480 A                                                                                                        \n",
       "481 A                                                                                                        \n",
       "482 T                                                                                                        \n",
       "    allele0 af    logl_H1    l_mle p_lrt       \n",
       "1   A       0.105 -10.468100 1e+05 0.45864360  \n",
       "2   G       0.105 -10.468100 1e+05 0.45864360  \n",
       "3   G       0.763 -10.645140 1e+05 0.65867480  \n",
       "4   C       0.184 -10.119090 1e+05 0.26408160  \n",
       "5   G       0.105 -10.468100 1e+05 0.45864360  \n",
       "6   T       0.105 -10.468100 1e+05 0.45864360  \n",
       "7   T       0.105 -10.468100 1e+05 0.45864360  \n",
       "8   G       0.105 -10.468100 1e+05 0.45864360  \n",
       "9   T       0.263 -10.732210 1e+05 0.88480080  \n",
       "10  CA      0.500 -10.688950 1e+05 0.74300190  \n",
       "11  T       0.500 -10.688950 1e+05 0.74300190  \n",
       "12  G       0.263 -10.732210 1e+05 0.88480080  \n",
       "13  C       0.763 -10.645140 1e+05 0.65867480  \n",
       "14  C       0.053  -9.441112 1e+05 0.10664900  \n",
       "15  C       0.053  -9.441112 1e+05 0.10664900  \n",
       "16  T       0.053  -9.441112 1e+05 0.10664900  \n",
       "17  C       0.053 -10.603440 1e+05 0.59765860  \n",
       "18  A       0.737 -10.576860 1e+05 0.56466830  \n",
       "19  A       0.579  -8.153650 1e+05 0.02287315  \n",
       "20  C       0.184  -9.285249 1e+05 0.08776407  \n",
       "21  G       0.053  -8.393842 1e+05 0.03020251  \n",
       "22  G       0.684  -9.205706 1e+05 0.07955344  \n",
       "23  A       0.079  -8.037921 1e+05 0.02002664  \n",
       "24  G       0.289 -10.590850 1e+05 0.58156830  \n",
       "25  G       0.053 -10.696080 1e+05 0.76007180  \n",
       "26  G       0.053 -10.300630 1e+05 0.34706420  \n",
       "27  C       0.053 -10.300630 1e+05 0.34706420  \n",
       "28  T       0.105 -10.731110 1e+05 0.87897270  \n",
       "29  C       0.053 -10.300630 1e+05 0.34706420  \n",
       "30  G       0.289 -10.590850 1e+05 0.58156830  \n",
       "⋮   ⋮       ⋮     ⋮          ⋮     ⋮           \n",
       "453 A       0.079 -10.169340 1e+05 0.2842344000\n",
       "454 C       0.105 -10.658940 1e+05 0.6823087000\n",
       "455 A       0.132  -9.586946 1e+05 0.1284184000\n",
       "456 G       0.289 -10.606420 1e+05 0.6016047000\n",
       "457 C       0.158 -10.263390 1e+05 0.3275301000\n",
       "458 C       0.079  -8.244272 1e+05 0.0253932200\n",
       "459 G       0.053 -10.714880 1e+05 0.8134935000\n",
       "460 G       0.105 -10.646590 1e+05 0.6610606000\n",
       "461 C       0.263 -10.114500 1e+05 0.2623302000\n",
       "462 A       0.105  -8.552956 1e+05 0.0363736500\n",
       "463 A       0.947 -10.696080 1e+05 0.7600718000\n",
       "464 G       0.105  -8.552956 1e+05 0.0363736500\n",
       "465 CAT     0.316  -5.599546 1e+05 0.0013402000\n",
       "466 C       0.447 -10.624620 1e+05 0.6269828000\n",
       "467 A       0.053 -10.696080 1e+05 0.7600718000\n",
       "468 A       0.368 -10.199840 1e+05 0.2974179000\n",
       "469 C       0.105  -9.005470 1e+05 0.0623225500\n",
       "470 G       0.105  -9.005470 1e+05 0.0623225500\n",
       "471 A       0.921 -10.096590 1e+05 0.2556367000\n",
       "472 A       0.158 -10.626240 1e+05 0.6293590000\n",
       "473 A       0.895 -10.556750 1e+05 0.5419614000\n",
       "474 T       0.737  -9.747092 1e+05 0.1582126000\n",
       "475 G       0.737  -9.747092 1e+05 0.1582126000\n",
       "476 G       0.342  -5.279159 1e+05 0.0009476796\n",
       "477 G       0.184 -10.735120 1e+05 0.9019485000\n",
       "478 TA      0.184 -10.735120 1e+05 0.9019485000\n",
       "479 A       0.184 -10.735120 1e+05 0.9019485000\n",
       "480 G       0.184 -10.735120 1e+05 0.9019485000\n",
       "481 C       0.158 -10.626240 1e+05 0.6293590000\n",
       "482 TGATA   0.053  -9.303518 1e+05 0.0897765000"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "resultGEMMA"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bca4d783-8578-4d12-946c-9a540a356feb",
   "metadata": {},
   "source": [
    "## Results visualization"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "c3c5564a-0af1-4f0c-8b49-e5512d5d4ca3",
   "metadata": {},
   "outputs": [],
   "source": [
    "bonferroni <- log10(0.05/row(resultGEMMA))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "ff7dc339-f484-436e-974e-3d8b52ac2f5b",
   "metadata": {},
   "outputs": [
    {
     "ename": "ERROR",
     "evalue": "Error in if (genomewideline) abline(h = genomewideline, col = \"red\"): the condition has length > 1\n",
     "output_type": "error",
     "traceback": [
      "Error in if (genomewideline) abline(h = genomewideline, col = \"red\"): the condition has length > 1\nTraceback:\n",
      "1. manhattan(resultGEMMA, chr = \"chr\", bp = \"ps\", p = \"p_lrt\", snp = \"rs\", \n .     genomewideline = bonferroni)"
     ]
    }
   ],
   "source": [
    "png(\"GWASresults_manhattan.png\")\n",
    "manhattan(resultGEMMA, chr=\"chr\", bp=\"ps\", p=\"p_lrt\", snp=\"rs\", genomewideline = bonferroni)\n",
    "dev.off()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "ccefd2e8-e401-4e6f-b71c-8cb302f2972d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<strong>png:</strong> 2"
      ],
      "text/latex": [
       "\\textbf{png:} 2"
      ],
      "text/markdown": [
       "**png:** 2"
      ],
      "text/plain": [
       "png \n",
       "  2 "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "## QQ plot\n",
    "png(\"GWAS_casecontrol_opiod_QQplot.png\")\n",
    "    qq(resultGEMMA$p_lrt)\n",
    "dev.off()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "56306337-7d53-429f-b2b8-37499a5b262c",
   "metadata": {},
   "source": [
    "## Extract top results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "e95d72e1-835c-4910-bd21-17b5ba095950",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class=\"dataframe\">\n",
       "<caption>A tibble: 0 × 4</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>chr</th><th scope=col>rs</th><th scope=col>p_lrt</th><th scope=col>negLogP</th></tr>\n",
       "\t<tr><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A tibble: 0 × 4\n",
       "\\begin{tabular}{llll}\n",
       " chr & rs & p\\_lrt & negLogP\\\\\n",
       " <dbl> & <chr> & <dbl> & <dbl>\\\\\n",
       "\\hline\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A tibble: 0 × 4\n",
       "\n",
       "| chr &lt;dbl&gt; | rs &lt;chr&gt; | p_lrt &lt;dbl&gt; | negLogP &lt;dbl&gt; |\n",
       "|---|---|---|---|\n",
       "\n"
      ],
      "text/plain": [
       "     chr rs p_lrt negLogP"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "resultGEMMA %>%\n",
    "    mutate(negLogP = -log10(p_lrt)) %>%\n",
    "    select(chr, rs, p_lrt, negLogP) %>%\n",
    "    filter(negLogP > 5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a5e88f3e-db07-49ee-9b9e-7fd635131460",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R",
   "language": "R",
   "name": "ir"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "4.3.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
